// random-movies/dist/chunk-6PVQY675.mjs
// src/data/movies.ts
var movies = [
  {
    title: "The Shawshank Redemption",
    description: "Framed in the 1940s for the double murder of his wife and her lover, upstanding banker Andy Dufresne begins a new life at the Shawshank prison, where he puts his accounting skills to work for an amoral warden. During his long stretch in prison, Dufresne comes to be admired by the other inmates -- including an older prisoner named Red -- for his integrity and unquenchable sense of hope."
  },
  {
    title: "Dilwale Dulhania Le Jayenge",
    description: "Raj is a rich, carefree, happy-go-lucky second generation NRI. Simran is the daughter of Chaudhary Baldev Singh, who in spite of being an NRI is very strict about adherence to Indian values. Simran has left for India to be married to her childhood fianc\xE9. Raj leaves for India with a mission at his hands, to claim his lady love under the noses of her whole family. Thus begins a saga."
  },
  {
    title: "The Godfather",
    description: "Spanning the years 1945 to 1955, a chronicle of the fictional Italian-American Corleone crime family. When organized crime family patriarch, Vito Corleone barely survives an attempt on his life, his youngest son, Michael steps in to take care of the would-be killers, launching a campaign of bloody revenge."
  },
  {
    title: "Schindler's List",
    description: "The true story of how businessman Oskar Schindler saved over a thousand Jewish lives from the Nazis while they worked as slaves in his factory during World War II."
  },
  {
    title: "The Godfather: Part II",
    description: "In the continuing saga of the Corleone crime family, a young Vito Corleone grows up in Sicily and in 1910s New York. In the 1950s, Michael Corleone attempts to expand the family business into Las Vegas, Hollywood and Cuba."
  },
  {
    title: "Impossible Things",
    description: "A widow who is tormented by the memory of her abusive husband befriends a young man."
  },
  {
    title: "Spirited Away",
    description: "A young girl, Chihiro, becomes trapped in a strange new world of spirits. When her parents undergo a mysterious transformation, she must call upon the courage she never knew she had to free her family."
  },
  {
    title: "Dou kyu sei \u2013 Classmates",
    description: "Rihito Sajo, an honor student with a perfect score on the entrance exam and Hikaru Kusakabe, in a band and popular among girls, would have never crossed paths. Until one day they started talking at the practice for their school\u2019s upcoming chorus festival. After school, the two meet regularly, as Hikaru helps Rihito to improve his singing skills. While they listen to each other\u2019s voice and harmonize, their hearts start to beat together."
  },
  {
    title: "Your Name.",
    description: "High schoolers Mitsuha and Taki are complete strangers living separate lives. But one night, they suddenly switch places. Mitsuha wakes up in Taki\u2019s body, and he in hers. This bizarre occurrence continues to happen randomly, and the two must adjust their lives around each other."
  },
  {
    title: "Your Eyes Tell",
    description: "A tragic accident lead to Kaori's blindness, but she clings to life and the smaller pleasures it can still afford her. She meets Rui and begins to talk to him. Rui was once a promising kickboxer, but something happened in his past. Kaori's smile brings out a change in Rui. However, the two are connected in more than one way. Rui attempts to do what is right."
  },
  {
    title: "12 Angry Men",
    description: "The defense and the prosecution have rested and the jury is filing into the jury room to decide if a young Spanish-American is guilty or innocent of murdering his father. What begins as an open and shut case soon becomes a mini-drama of each of the jurors' prejudices and preconceptions about the trial, the accused, and each other."
  },
  {
    title: "Gabriel's Inferno",
    description: "An intriguing and sinful exploration of seduction, forbidden love, and redemption, Gabriel's Inferno is a captivating and wildly passionate tale of one man's escape from his own personal hell as he tries to earn the impossible--forgiveness and love."
  },
  {
    title: "Parasite",
    description: "All unemployed, Ki-taek's family takes peculiar interest in the wealthy and glamorous Parks for their livelihood until they get entangled in an unexpected incident."
  },
  {
    title: "The Green Mile",
    description: "A supernatural tale set on death row in a Southern prison, where gentle giant John Coffey possesses the mysterious power to heal people's ailments. When the cell block's head guard, Paul Edgecomb, recognizes Coffey's miraculous gift, he tries desperately to help stave off the condemned man's execution."
  },
  {
    title: "Gabriel's Inferno: Part II",
    description: "Professor Gabriel Emerson finally learns the truth about Julia Mitchell's identity, but his realization comes a moment too late. Julia is done waiting for the well-respected Dante specialist to remember her and wants nothing more to do with him. Can Gabriel win back her heart before she finds love in another's arms?"
  },
  {
    title: "The Dark Knight",
    description: "Batman raises the stakes in his war on crime. With the help of Lt. Jim Gordon and District Attorney Harvey Dent, Batman sets out to dismantle the remaining criminal organizations that plague the streets. The partnership proves to be effective, but they soon find themselves prey to a reign of chaos unleashed by a rising criminal mastermind known to the terrified citizens of Gotham as the Joker."
  },
  {
    title: "The Good, the Bad and the Ugly",
    description: "While the Civil War rages between the Union and the Confederacy, three men \u2013 a quiet loner, a ruthless hit man and a Mexican bandit \u2013 comb the American Southwest in search of a strongbox containing $200,000 in stolen gold."
  },
  {
    title: "Pulp Fiction",
    description: "A burger-loving hit man, his philosophical partner, a drug-addled gangster's moll and a washed-up boxer converge in this sprawling, comedic crime caper. Their adventures unfurl in three stories that ingeniously trip back and forth in time."
  },
  {
    title: "The Lord of the Rings: The Return of the King",
    description: "Aragorn is revealed as the heir to the ancient kings as he, Gandalf and the other members of the broken fellowship struggle to save Gondor from Sauron's forces. Meanwhile, Frodo and Sam take the ring closer to the heart of Mordor, the dark lord's realm."
  },
  {
    title: "Gabriel's Inferno: Part III",
    description: "The final part of the film adaption of the erotic romance novel Gabriel's Inferno written by an anonymous Canadian author under the pen name Sylvain Reynard."
  },
  {
    title: "Violet Evergarden: The Movie",
    description: "As the world moves on from the war and technological advances bring changes to her life, Violet still hopes to see her lost commanding officer again."
  },
  {
    title: "Forrest Gump",
    description: "A man with a low IQ has accomplished great things in his life and been present during significant historic events\u2014in each case, far exceeding what anyone imagined he could do. But despite all he has achieved, his one true love eludes him."
  },
  {
    title: "Cinema Paradiso",
    description: "A filmmaker recalls his childhood, when he fell in love with the movies at his village's theater and formed a deep friendship with the theater's projectionist."
  },
  {
    title: "Seven Samurai",
    description: "A samurai answers a village's request for protection after he falls on hard times. The town needs protection from bandits, so the samurai gathers six others to help him teach the people how to defend themselves, and the villagers provide the soldiers with food."
  },
  {
    title: "Josee, the Tiger and the Fish",
    description: "With dreams of diving abroad, Tsuneo gets a job assisting Josee, an artist whose imagination takes her far beyond her wheelchair. But when the tide turns against them, they push each other to places they never thought possible, and inspire a love fit for a storybook."
  },
  {
    title: "GoodFellas",
    description: "The true story of Henry Hill, a half-Irish, half-Sicilian Brooklyn kid who is adopted by neighbourhood gangsters at an early age and climbs the ranks of a Mafia family under the guidance of Jimmy Conway."
  },
  {
    title: "Life Is Beautiful",
    description: "A touching story of an Italian book seller of Jewish ancestry who lives in his own little fairy tale. His creative and happy life would come to an abrupt halt when his entire family is deported to a concentration camp during World War II. While locked up he tries to convince his son that the whole thing is just a game."
  },
  {
    title: "Once Upon a Time in America",
    description: "A former Prohibition-era Jewish gangster returns to the Lower East Side of Manhattan over thirty years later, where he once again must confront the ghosts and regrets of his old life."
  },
  {
    title: "Harakiri",
    description: "Down-on-his-luck veteran Tsugumo Hanshir\u014D enters the courtyard of the prosperous House of Iyi. Unemployed, and with no family, he hopes to find a place to commit seppuku\u2014and a worthy second to deliver the coup de gr\xE2ce in his suicide ritual. The senior counselor for the Iyi clan questions the ronin\u2019s resolve and integrity, suspecting Hanshir\u014D of seeking charity rather than an honorable end. What follows is a pair of interlocking stories which lay bare the difference between honor and respect, and promises to examine the legendary foundations of the Samurai code."
  },
  {
    title: "Psycho",
    description: "When larcenous real estate clerk Marion Crane goes on the lam with a wad of cash and hopes of starting a new life, she ends up at the notorious Bates Motel, where manager Norman Bates cares for his housebound mother."
  },
  {
    title: "Grave of the Fireflies",
    description: "In the final months of World War II, 14-year-old Seita and his sister Setsuko are orphaned when their mother is killed during an air raid in Kobe, Japan. After a falling out with their aunt, they move into an abandoned bomb shelter. With no surviving relatives and their emergency rations depleted, Seita and Setsuko struggle to survive."
  },
  {
    title: "A Dog's Will",
    description: "The lively Jack the Cricket and the sly Chic\xF3 are poor guys living in the hinterland who cheat a bunch of people in a small in Northeastern Brazil. When they die, they have to be judged by Christ, the Devil and the Virgin Mary before they are admitted to paradise."
  },
  {
    title: "One Flew Over the Cuckoo's Nest",
    description: "A petty criminal fakes insanity to serve his sentence in a mental ward rather than prison. He soon finds himself as a leader to the other patients\u2014and an enemy to the cruel, domineering nurse who runs the ward."
  },
  {
    title: "Fight Club",
    description: 'A ticking-time-bomb insomniac and a slippery soap salesman channel primal male aggression into a shocking new form of therapy. Their concept catches on, with underground "fight clubs" forming in every town, until an eccentric gets in the way and ignites an out-of-control spiral toward oblivion.'
  },
  {
    title: "Evangelion: 3.0+1.0 Thrice Upon a Time",
    description: "In the aftermath of the Fourth Impact, stranded without their Evangelions, Shinji, Asuka, and Rei find refuge in one of the rare pockets of humanity that still exist on the ruined planet Earth. There, each of them live a life far different from their days as an Evangelion pilot. However, the danger to the world is far from over. A new impact is looming on the horizon\u2014one that will prove to be the true end of Evangelion."
  },
  {
    title: "Spider-Man: Into the Spider-Verse",
    description: 'Miles Morales is juggling his life between being a high school student and being a spider-man. When Wilson "Kingpin" Fisk uses a super collider, others from across the Spider-Verse are transported to this dimension.'
  },
  {
    title: "City of God",
    description: "Buscap\xE9 was raised in a very violent environment. Despite the feeling that all odds were against him, he finds out that life can be seen with other eyes..."
  },
  {
    title: "Hope",
    description: "After 8-year-old So-won narrowly survives a brutal sexual assault, her family labors to help her heal while coping with their own rage and grief."
  },
  {
    title: "A Silent Voice: The Movie",
    description: "Shouya Ishida starts bullying the new girl in class, Shouko Nishimiya, because she is deaf. But as the teasing continues, the rest of the class starts to turn on Shouya for his lack of compassion. When they leave elementary school, Shouko and Shouya do not speak to each other again... until an older, wiser Shouya, tormented by his past behaviour, decides he must see Shouko once more. He wants to atone for his sins, but is it already too late...?"
  },
  {
    title: "Hotarubi no Mori e",
    description: "One hot summer day a little girl gets lost in an enchanted forest of the mountain god where spirits reside. A young boy appears before her, but she cannot touch him for fear of making him disappear. And so a wondrous adventure awaits..."
  },
  {
    title: "Howl's Moving Castle",
    description: "When Sophie, a shy young woman, is cursed with an old body by a spiteful witch, her only chance of breaking the spell lies with a self-indulgent yet insecure young wizard and his companions in his legged, walking castle."
  },
  {
    title: "Neon Genesis Evangelion: The End of Evangelion",
    description: "The second of two theatrically released follow-ups to the Neon Genesis Evangelion series. Comprising of two alternate episodes which were first intended to take the place of episodes 25 and 26, this finale answers many of the questions surrounding the series, while also opening up some new possibilities."
  },
  {
    title: "The Empire Strikes Back",
    description: "The epic saga continues as Luke Skywalker, in hopes of defeating the evil Galactic Empire, learns the ways of the Jedi from aging master Yoda. But Darth Vader is more determined than ever to capture Luke. Meanwhile, rebel leader Princess Leia, cocky Han Solo, Chewbacca, and droids C-3PO and R2-D2 are thrown into various stages of capture, betrayal and despair."
  },
  {
    title: "Sunset Boulevard",
    description: "A hack screenwriter writes a screenplay for a former silent film star who has faded into Hollywood obscurity."
  },
  {
    title: "Wolfwalkers",
    description: "In a time of superstition and magic, when wolves are seen as demonic and nature an evil to be tamed, a young apprentice hunter comes to Ireland with her father to wipe out the last pack. But when she saves a wild native girl, their friendship leads her to discover the world of the Wolfwalkers and transform her into the very thing her father is tasked to destroy."
  },
  {
    title: "The Lord of the Rings: The Fellowship of the Ring",
    description: "Young hobbit Frodo Baggins, after inheriting a mysterious ring from his uncle Bilbo, must leave his home in order to keep it from falling into the hands of its evil creator. Along the way, a fellowship is formed to protect the ringbearer and make sure that the ring arrives at its final destination: Mt. Doom, the only place where it can be destroyed."
  },
  {
    title: "Primal: Tales of Savagery",
    description: "Genndy Tartakovsky's Primal: Tales of Savagery features a caveman and a dinosaur on the brink of extinction. Bonded by tragedy, this unlikely friendship becomes the only hope of survival."
  },
  {
    title: "Ikiru",
    description: "Kanji Watanabe is a middle-aged man who has worked in the same monotonous bureaucratic position for decades. Learning he has cancer, he starts to look for the meaning of his life."
  },
  {
    title: "The Pianist",
    description: "The true story of pianist W\u0142adys\u0142aw Szpilman's experiences in Warsaw during the Nazi occupation. When the Jews of the city find themselves forced into a ghetto, Szpilman finds work playing in a caf\xE9; and when his family is deported in 1942, he stays behind, works for a while as a laborer, and eventually goes into hiding in the ruins of the war-torn city."
  },
  {
    title: "Interstellar",
    description: "The adventures of a group of explorers who make use of a newly discovered wormhole to surpass the limitations on human space travel and conquer the vast distances involved in an interstellar voyage."
  },
  {
    title: "Whiplash",
    description: "Under the direction of a ruthless instructor, a talented young drummer begins to pursue perfection at any cost, even his humanity."
  },
  {
    title: "The Lord of the Rings: The Two Towers",
    description: "Frodo and Sam are trekking to Mordor to destroy the One Ring of Power while Gimli, Legolas and Aragorn search for the orc-captured Merry and Pippin. All along, nefarious wizard Saruman awaits the Fellowship members at the Orthanc Tower in Isengard."
  },
  {
    title: "American History X",
    description: "Derek Vineyard is paroled after serving 3 years in prison for killing two African-American men. Through his brother, Danny Vineyard's narration, we learn that before going to prison, Derek was a skinhead and the leader of a violent white supremacist gang that committed acts of racial crime throughout L.A. and his actions greatly influenced Danny. Reformed and fresh out of prison, Derek severs contact with the gang and becomes determined to keep Danny from going down the same violent path as he did."
  },
  {
    title: "Rear Window",
    description: "A wheelchair-bound photographer spies on his neighbors from his apartment window and becomes convinced one of them has committed murder."
  },
  {
    title: "Inception",
    description: `Cobb, a skilled thief who commits corporate espionage by infiltrating the subconscious of his targets is offered a chance to regain his old life as payment for a task considered to be impossible: "inception", the implantation of another person's idea into a target's subconscious.`
  },
  {
    title: "Demon Slayer -Kimetsu no Yaiba- The Movie: Mugen Train",
    description: "Tanjir\u014D Kamado, joined with Inosuke Hashibira, a boy raised by boars who wears a boar's head, and Zenitsu Agatsuma, a scared boy who reveals his true power when he sleeps, boards the Infinity Train on a new mission with the Fire Hashira, Ky\u014Djur\u014D Rengoku, to defeat a demon who has been tormenting the people and killing the demon slayers who oppose it!"
  },
  {
    title: "Given",
    description: "The relationship between a band's bassist, their drummer, and the drummer\u2019s roommate and ex-boyfriend, who is a professional violinist."
  },
  {
    title: "City Lights",
    description: "In this sound-era silent film, a tramp falls in love with a beautiful blind flower seller."
  },
  {
    title: "The Great Dictator",
    description: "Dictator Adenoid Hynkel tries to expand his empire while a poor Jewish barber tries to avoid persecution from Hynkel's regime."
  },
  {
    title: "Se7en",
    description: `Two homicide detectives are on a desperate hunt for a serial killer whose crimes are based on the "seven deadly sins" in this dark and haunting film that takes viewers from the tortured remains of one victim to the next. The seasoned Det. Sommerset researches each sin in an effort to get inside the killer's mind, while his novice partner, Mills, scoffs at his efforts to unravel the case.`
  },
  {
    title: "High and Low",
    description: "An executive of a shoe company becomes a victim of extortion when his chauffeur's son is kidnapped and held for ransom."
  },
  {
    title: "Top Gun: Maverick",
    description: "After more than thirty years of service as one of the Navy\u2019s top aviators, and dodging the advancement in rank that would ground him, Pete \u201CMaverick\u201D Mitchell finds himself training a detachment of TOP GUN graduates for a specialized mission the likes of which no living pilot has ever seen."
  },
  {
    title: "Princess Mononoke",
    description: "Ashitaka, a prince of the disappearing Emishi people, is cursed by a demonized boar god and must journey to the west to find a cure. Along the way, he encounters San, a young human woman fighting to protect the forest, and Lady Eboshi, who is trying to destroy it. Ashitaka must find a way to bring balance to this conflict."
  },
  {
    title: "Dedicated to my ex",
    description: "The film tells the story of Ariel, a 21-year-old who decides to form a rock band to compete for a prize of ten thousand dollars in a musical band contest, this as a last option when trying to get money to save their relationship and reunite with his ex-girlfriend, which breaks due to the trip she must make to Finland for an internship. Ariel with her friend Ortega, decides to make a casting to find the other members of the band, although they do not know nothing about music, thus forming a band with members that have diverse and opposite personalities."
  },
  {
    title: "The Silence of the Lambs",
    description: "Clarice Starling is a top student at the FBI's training academy. Jack Crawford wants Clarice to interview Dr. Hannibal Lecter, a brilliant psychiatrist who is also a violent psychopath, serving life behind bars for various acts of murder and cannibalism. Crawford believes that Lecter may have insight into a case and that Starling, as an attractive young woman, may be just the bait to draw him out."
  },
  {
    title: "Come and See",
    description: "The invasion of a village in Byelorussia by German forces sends young Florya into the forest to join the weary Resistance fighters, against his family's wishes. There he meets a girl, Glasha, who accompanies him back to his village. On returning home, Florya finds his family and fellow peasants massacred. His continued survival amidst the brutal debris of war becomes increasingly nightmarish, a battle between despair and hope."
  },
  {
    title: "Clouds",
    description: "Young musician Zach Sobiech discovers his cancer has spread, leaving him just a few months to live. With limited time, he follows his dream and makes an album, unaware that it will soon be a viral music phenomenon."
  },
  {
    title: "Modern Times",
    description: "The Tramp struggles to live in modern industrial society with the help of a young homeless woman."
  },
  {
    title: "Dead Poets Society",
    description: "At an elite, old-fashioned boarding school in New England, a passionate English teacher inspires his students to rebel against convention and seize the potential of every day, courting the disdain of the stern headmaster."
  },
  {
    title: "Five Feet Apart",
    description: "Seventeen-year-old Stella spends most of her time in the hospital as a cystic fibrosis patient. Her life is full of routines, boundaries and self-control \u2014 all of which get put to the test when she meets Will, an impossibly charming teen who has the same illness. There's an instant flirtation, though restrictions dictate that they must maintain a safe distance between them. As their connection intensifies, so does the temptation to throw the rules out the window and embrace that attraction."
  },
  {
    title: "L\xE9on: The Professional",
    description: `L\xE9on, the top hit man in New York, has earned a rep as an effective "cleaner". But when his next-door neighbors are wiped out by a loose-cannon DEA agent, he becomes the unwilling custodian of 12-year-old Mathilda. Before long, Mathilda's thoughts turn to revenge, and she considers following in L\xE9on's footsteps.`
  },
  {
    title: "Once Upon a Time in the West",
    description: "As the railroad builders advance unstoppably through the Arizona desert on their way to the sea, Jill arrives in the small town of Flagstone with the intention of starting a new life."
  },
  {
    title: "Hamilton",
    description: "Presenting the tale of American founding father Alexander Hamilton, this filmed version of the original Broadway smash hit is the story of America then, told by America now."
  },
  {
    title: "Paths of Glory",
    description: "A commanding officer defends three scapegoats on trial for a failed offensive that occurred within the French Army in 1916."
  },
  {
    title: "Back to the Future",
    description: "Eighties teenager Marty McFly is accidentally sent back in time to 1955, inadvertently disrupting his parents' first meeting and attracting his mother's romantic interest. Marty must repair the damage to history by rekindling his parents' romance and - with the help of his eccentric inventor friend Doc Brown - return to 1985."
  },
  {
    title: "Justice League Dark: Apokolips War",
    description: "Earth is decimated after intergalactic tyrant Darkseid has devastated the Justice League in a poorly executed war by the DC Super Heroes. Now the remaining bastions of good \u2013 the Justice League, Teen Titans, Suicide Squad and assorted others \u2013 must regroup, strategize and take the war to Darkseid in order to save the planet and its surviving inhabitants."
  },
  {
    title: "My Hero Academia: Heroes Rising",
    description: "Class 1-A visits Nabu Island where they finally get to do some real hero work. The place is so peaceful that it's more like a vacation \u2026 until they're attacked by a villain with an unfathomable Quirk! His power is eerily familiar, and it looks like Shigaraki had a hand in the plan. But with All Might retired and citizens' lives on the line, there's no time for questions. Deku and his friends are the next generation of heroes, and they're the island's only hope."
  },
  {
    title: "Life in a Year",
    description: "A 17 year old finds out that his girlfriend is dying, so he sets out to give her an entire life, in the last year she has left."
  },
  {
    title: "Woman in the Dunes",
    description: "An entomologist suffers extreme psychological and sexual torture after being taken captive by the residents of a poor seaside village."
  },
  {
    title: "Better Days",
    description: "A bullied teenage girl forms an unlikely friendship with a mysterious young man who protects her from her assailants, while she copes with the pressures of her final examinations."
  },
  {
    title: "We All Loved Each Other So Much",
    description: "Three partisans bound by a strong friendship return home after the war, but the clash with everyday reality puts a strain on their bond."
  },
  {
    title: "Tokyo Story",
    description: "The elderly Shukishi and his wife, Tomi, take the long journey from their small seaside village to visit their adult children in Tokyo. Their elder son, Koichi, a doctor, and their daughter, Shige, a hairdresser, don't have much time to spend with their aged parents, and so it falls to Noriko, the widow of their younger son who was killed in the war, to keep her in-laws company."
  },
  {
    title: "Perfect Blue",
    description: "A retired pop singer turned actress' sense of reality is shaken when she is stalked by an obsessed fan and seemingly a ghost of her past."
  },
  {
    title: "The Legend of 1900",
    description: "The story of a virtuoso piano player who lives his entire life aboard an ocean liner. Born and raised on the ship, 1900 (Tim Roth) learned about the outside world through interactions with passengers, never setting foot on land, even for the love of his life. Years later, the ship may be destroyed, and a former band member fears that 1900 may still be aboard, willing to go down with the ship."
  },
  {
    title: "Apocalypse Now",
    description: 'At the height of the Vietnam war, Captain Benjamin Willard is sent on a dangerous mission that, officially, "does not exist, nor will it ever exist." His goal is to locate - and eliminate - a mysterious Green Beret Colonel named Walter Kurtz, who has been leading his personal army on illegal guerrilla missions into enemy territory.'
  },
  {
    title: "Rascal Does Not Dream of a Dreaming Girl",
    description: "In Fujisawa, Sakuta Azusagawa is in his second year of high school. Blissful days with his girlfriend and upperclassman, Mai Sakurajima, are interrupted by the appearance of his first crush, Shoko Makinohara."
  },
  {
    title: "Avengers: Endgame",
    description: "After the devastating events of Avengers: Infinity War, the universe is in ruins due to the efforts of the Mad Titan, Thanos. With the help of remaining allies, the Avengers must assemble once more in order to undo Thanos' actions and restore order to the universe once and for all, no matter what consequences may be in store."
  },
  {
    title: "Le Trou",
    description: "Four prison inmates have been hatching a plan to literally dig out of jail when another prisoner, Claude Gaspard, is moved into their cell. They take a risk and share their plan with the newcomer. Over the course of three days, the prisoners and friends break through the concrete floor using a bed post and begin to make their way through the sewer system -- yet their escape is anything but assured."
  },
  {
    title: "Miracle in Cell No. 7",
    description: "Separated from his daughter, a father with an intellectual disability must prove his innocence when he is jailed for the death of a commander's child."
  },
  {
    title: "Mommy",
    description: "A peculiar neighbor offers hope to a recent widow who is struggling to raise a teenager who is unpredictable and, sometimes, violent."
  },
  {
    title: "Oldboy",
    description: "With no clue how he came to be imprisoned, drugged and tortured for 15 years, a desperate businessman seeks revenge on his captors."
  },
  {
    title: "Steven Universe: The Movie",
    description: "Two years after bringing peace to the galaxy, Steven Universe sees his past come back to haunt him in the form of a deranged Gem who wants to destroy the Earth."
  },
  {
    title: "Everything Everywhere All at Once",
    description: "An aging Chinese immigrant is swept up in an insane adventure, where she alone can save what's important to her by connecting with the lives she could have led in other universes."
  },
  {
    title: "The Art of Racing in the Rain",
    description: "A family dog - with a near-human soul and a philosopher's mind - evaluates his life through the lessons learned by his human owner, a race-car driver."
  },
  {
    title: "Zack Snyder's Justice League",
    description: "Determined to ensure Superman's ultimate sacrifice was not in vain, Bruce Wayne aligns forces with Diana Prince with plans to recruit a team of metahumans to protect the world from an approaching threat of catastrophic proportions."
  },
  {
    title: "The Handmaiden",
    description: "A Japanese woman hires a handmaiden who is secretly working with a con man to cheat her out of her wealth."
  },
  {
    title: "Klaus",
    description: "When Jesper distinguishes himself as the Postal Academy's worst student, he is sent to Smeerensburg, a small village located on an icy island above the Arctic Circle, where grumpy inhabitants barely exchange words, let alone letters. Jesper is about to give up and abandon his duty as a postman when he meets local teacher Alva and Klaus, a mysterious carpenter who lives alone in a cabin full of handmade toys."
  },
  {
    title: "Avengers: Infinity War",
    description: "As the Avengers and their allies have continued to protect the world from threats too large for any one hero to handle, a new danger has emerged from the cosmic shadows: Thanos. A despot of intergalactic infamy, his goal is to collect all six Infinity Stones, artifacts of unimaginable power, and use them to inflict his twisted will on all of reality. Everything the Avengers have fought for has led up to this moment - the fate of Earth and existence itself has never been more uncertain."
  },
  {
    title: "The Intouchables",
    description: "A true story of two men who should never have met \u2013 a quadriplegic aristocrat who was injured in a paragliding accident and a young man from the projects."
  },
  {
    title: "KonoSuba: God's Blessing on this Wonderful World! Legend of Crimson",
    description: "It is not strange that the Demon Lord's forces fear the Crimson Demons, the clan from which Megumin and Yunyun originate. Even if the Demon Lord's generals attack their village, the Crimson Demons can just easily brush them off with their supreme mastery of advanced and overpowered magic.  When Yunyun receives a seemingly serious letter regarding a potential disaster coming to her hometown, she immediately informs Kazuma Satou and the rest of his party. After a series of wacky misunderstandings, it turns out to be a mere prank by her fellow demon who wants to be an author. Even so, Megumin becomes worried about her family and sets out toward the Crimson Demons' village with the gang.  There, Kazuma and the others decide to sightsee the wonders of Megumin's birthplace. However, they soon come to realize that the nonsense threat they received might have been more than just a joke."
  },
  {
    title: "The Lion King",
    description: "A young lion prince is cast out of his pride by his cruel uncle, who claims he killed his father. While the uncle rules with an iron paw, the prince grows up beyond the Savannah, living by a philosophy: No worries for the rest of your days. But when his past comes to haunt him, the young prince must decide his fate: Will he remain an outcast or face his demons and become what he needs to be?"
  },
  {
    title: "It's a Wonderful Life",
    description: "A holiday favourite for generations...  George Bailey has spent his entire life giving to the people of Bedford Falls.  All that prevents rich skinflint Mr. Potter from taking over the entire town is George's modest building and loan company.  But on Christmas Eve the business's $8,000 is lost and George's troubles begin."
  },
  {
    title: "Persona",
    description: "A young nurse, Alma, is put in charge of Elisabeth Vogler: an actress who is seemingly healthy in all respects, but will not talk. As they spend time together, Alma speaks to Elisabeth constantly, never receiving any answer. The time they spend together only strengthens the crushing realization that one does not exist."
  },
  {
    title: "Bicycle Thieves",
    description: "A working man's livelihood is threatened when someone steals his bicycle."
  },
  {
    title: "Black Beauty",
    description: "Born free in the American West, Black Beauty is a horse rounded up and brought to Birtwick Stables, where she meets spirited teenager Jo Green. The two forge a bond that carries Beauty through the different chapters, challenges and adventures."
  },
  {
    title: "Maquia: When the Promised Flower Blooms",
    description: "Maquia is a member of a special race called the Iorph who can live for hundreds of years. However, Maquia has always felt lonely despite being surrounded by her people, as she was orphaned from a young age. She daydreams about the outside world, but dares not travel from her home due to the warnings of the clan's chief.  One day the kingdom of Mezarte invades her homeland. They already have what is left of the giant dragons, the Renato, under their control, and now their king wishes to add the immortality to his bloodline.  They ravage the Iorph homeland and kill most of its inhabitants. Caught in the midst of the attack, Maquia is carried off by one of the Renato. It soon dies, and she is left deserted in a forest, now truly alone save for the cries of a single baby off in the distance. Maquia finds the baby in a destroyed village and decides to raise him as her own, naming him Ariel. Although she knows nothing of the human world, how to raise a child that ages much faster than her."
  },
  {
    title: "Wolf Children",
    description: "After her werewolf lover unexpectedly dies in an accident, a woman must find a way to raise the son and daughter that she had with him. However, their inheritance of their father's traits prove to be a challenge for her."
  },
  {
    title: "Il Sorpasso",
    description: "Roberto, a shy law student in Rome, meets Bruno, a forty-year-old exuberant, capricious man, who takes him for a drive through the Roman and Tuscany countries in the summer. When their journey starts to blend into their daily lives though, the pair\u2019s newfound friendship is tested."
  },
  {
    title: "Green Book",
    description: "Tony Lip, a bouncer in 1962, is hired to drive pianist Don Shirley on a tour through the Deep South in the days when African Americans, forced to find alternate accommodations and services due to segregation laws below the Mason-Dixon Line, relied on a guide called The Negro Motorist Green Book."
  },
  {
    title: "Witness for the Prosecution",
    description: "When Leonard Vole is arrested for the sensational murder of a rich, middle-aged widow, the famous Sir Wilfrid Robarts agrees to appear on his behalf. Sir Wilfrid, recovering from a near-fatal heart attack, is supposed to be on a diet of bland, civil suits\u2014but the lure of the criminal courts is too much for him, especially when the case is so difficult."
  },
  {
    title: "Violet Evergarden: Eternity and the Auto Memory Doll",
    description: "Isabella, the daughter of the noble York family, is enrolled in an all-girls academy to be groomed into a dame worthy of nobility. However, she has given up on her future, seeing the prestigious school as nothing more than a prison from the outside world. Her family notices her struggling in her lessons and decides to hire Violet Evergarden to personally tutor her under the guise of a handmaiden.  At first, Isabella treats Violet coldly. Violet seems to be able to do everything perfectly, leading Isabella to assume that she was born with a silver spoon. After some time, Isabella begins to realize that Violet has had her own struggles and starts to open up to her. Isabella soon reveals that she has lost contact with her beloved younger sister, whom she yearns to see again.  Having experienced the power of words through her past clientele, Violet asks if Isabella wishes to write a letter to Taylor. Will Violet be able to help Isabella convey her feelings to her long-lost sister?"
  },
  {
    title: "I Want to Eat Your Pancreas",
    description: "After his classmate and crush is diagnosed with a pancreatic disease, an average high schooler sets out to make the most of her final days."
  },
  {
    title: "Sansho the Bailiff",
    description: "In medieval Japan a compassionate governor is sent into exile. His wife and children try to join him, but are separated, and the children grow up amid suffering and oppression."
  },
  {
    title: "Stalker",
    description: "Near a gray and unnamed city is the Zone, a place guarded by barbed wire and soldiers, and where the normal laws of physics are victim to frequent anomalies. A stalker guides two men into the Zone, specifically to an area in which deep-seated desires are granted."
  },
  {
    title: "Miraculous World: New York, United HeroeZ",
    description: "During a school field trip, Ladybug and Cat Noir meet the American superheroes, whom they have to save from an akumatised super-villain. They discover that Miraculous exist in the United States too."
  },
  {
    title: "Mortal Kombat Legends: Scorpion's Revenge",
    description: "After the vicious slaughter of his family by stone-cold mercenary Sub-Zero, Hanzo Hasashi is exiled to the torturous Netherrealm. There, in exchange for his servitude to the sinister Quan Chi, he\u2019s given a chance to avenge his family \u2013 and is resurrected as Scorpion, a lost soul bent on revenge. Back on Earthrealm, Lord Raiden gathers a team of elite warriors \u2013 Shaolin monk Liu Kang, Special Forces officer Sonya Blade and action star Johnny Cage \u2013 an unlikely band of heroes with one chance to save humanity. To do this, they must defeat Shang Tsung\u2019s horde of Outworld gladiators and reign over the Mortal Kombat tournament."
  },
  {
    title: "8\xBD",
    description: "Guido Anselmi, a film director, finds himself creatively barren at the peak of his career. Urged by his doctors to rest, Anselmi heads for a luxurious resort, but a sorry group gathers\u2014his producer, staff, actors, wife, mistress, and relatives\u2014each one begging him to get on with the show. In retreat from their dependency, he fantasizes about past women and dreams of his childhood."
  },
  {
    title: "The Kid",
    description: "A tramp cares for a boy after he's abandoned as a newborn by his mother. Later the mother has a change of heart and aches to be reunited with her son."
  },
  {
    title: "The Apartment",
    description: "Bud Baxter is a minor clerk in a huge New York insurance company, until he discovers a quick way to climb the corporate ladder. He lends out his apartment to the executives as a place to take their mistresses. Although he often has to deal with the aftermath of their visits, one night he's left with a major problem to solve."
  },
  {
    title: "Bo Burnham: Inside",
    description: "Stuck in COVID-19 lockdown, US comedian and musician Bo Burnham attempts to stay sane and happy by writing, shooting and performing a one-man comedy special."
  },
  {
    title: "New Gods: Nezha Reborn",
    description: "While living as an ordinary deliveryman and motor racing fan, Nezha encounters old nemeses and must rediscover his powers to protect his loved ones."
  },
  {
    title: "Coco",
    description: "Despite his family\u2019s baffling generations-old ban on music, Miguel dreams of becoming an accomplished musician like his idol, Ernesto de la Cruz. Desperate to prove his talent, Miguel finds himself in the stunning and colorful Land of the Dead following a mysterious chain of events. Along the way, he meets charming trickster Hector, and together, they set off on an extraordinary journey to unlock the real story behind Miguel's family history."
  },
  {
    title: "The Shining",
    description: "Jack Torrance accepts a caretaker job at the Overlook Hotel, where he, along with his wife Wendy and their son Danny, must live isolated from the rest of the world for the winter. But they aren't prepared for the madness that lurks within."
  },
  {
    title: "Doctor Who: The Day of the Doctor",
    description: "In 2013, something terrible is awakening in London's National Gallery; in 1562, a murderous plot is afoot in Elizabethan England; and somewhere in space an ancient battle reaches its devastating conclusion. All of reality is at stake as the Doctor's own dangerous past comes back to haunt him."
  },
  {
    title: "A Clockwork Orange",
    description: "In a near-future Britain, young Alexander DeLarge and his pals get their kicks beating and raping anyone they please. When not destroying the lives of others, Alex swoons to the music of Beethoven. The state, eager to crack down on juvenile crime, gives an incarcerated Alex the option to undergo an invasive procedure that'll rob him of all personal agency. In a time when conscience is a commodity, can Alex change his tune?"
  },
  {
    title: "The Seventh Seal",
    description: "When disillusioned Swedish knight Antonius Block returns home from the Crusades to find his country in the grips of the Black Death, he challenges Death to a chess match for his life. Tormented by the belief that God does not exist, Block sets off on a journey, meeting up with traveling players Jof and his wife, Mia, and becoming determined to evade Death long enough to commit one redemptive act while he still lives."
  },
  {
    title: "Investigation of a Citizen Above Suspicion",
    description: "Rome, Italy. After committing a heinous crime, a senior police officer exposes evidence incriminating him because his moral commitment prevents him from circumventing the law and the social order it protects."
  },
  {
    title: "Children of Paradise",
    description: "Filmed during the German occupation, this French milestone centers around the theatrical life of a beautiful courtesan and the four men who love her. Voted the \u201CBest French Film in History\u201D by the French Film Academy in 1990."
  },
  {
    title: "The Hate U Give",
    description: "Raised in a poverty-stricken slum, a 16-year-old girl named Starr now attends a suburban prep school. After she witnesses a police officer shoot her unarmed best friend, she's torn between her two very different worlds as she tries to speak her truth."
  },
  {
    title: "Vertigo",
    description: "A retired San Francisco detective suffering from acrophobia investigates the strange activities of an old friend's wife, all the while becoming dangerously obsessed with her."
  },
  {
    title: "Portrait of a Lady on Fire",
    description: "On an isolated island in Brittany at the end of the eighteenth century, a female painter is obliged to paint a wedding portrait of a young woman."
  },
  {
    title: "Inglourious Basterds",
    description: 'In Nazi-occupied France during World War II, a group of Jewish-American soldiers known as "The Basterds" are chosen specifically to spread fear throughout the Third Reich by scalping and brutally killing Nazis. The Basterds, lead by Lt. Aldo Raine soon cross paths with a French-Jewish teenage girl who runs a movie theater in Paris which is targeted by the soldiers.'
  },
  {
    title: "Call Me by Your Name",
    description: "In 1980s Italy, a relationship begins between seventeen-year-old teenage Elio and the older adult man hired as his father's research assistant."
  },
  {
    title: "Star Wars",
    description: "Princess Leia is captured and held hostage by the evil Imperial forces in their effort to take over the galactic Empire. Venturesome Luke Skywalker and dashing captain Han Solo team together with the loveable robot duo R2-D2 and C-3PO to rescue the beautiful princess and restore peace and justice in the Empire."
  },
  {
    title: "Along with the Gods: The Last 49 Days",
    description: "In the afterlife, one guardian helps a man through his trials, while his two colleagues help a former guardian on earth."
  },
  {
    title: "The Usual Suspects",
    description: "Held in an L.A. interrogation room, Verbal Kint attempts to convince the feds that a mythic crime lord, Keyser Soze, not only exists, but was also responsible for drawing him and his four partners into a multi-million dollar heist that ended with an explosion in San Pedro harbor \u2013 leaving few survivors. Verbal lures his interrogators with an incredible story of the crime lord's almost supernatural prowess."
  },
  {
    title: "Gladiator",
    description: "In the year 180, the death of emperor Marcus Aurelius throws the Roman Empire into chaos.  Maximus is one of the Roman army's most capable and trusted generals and a key advisor to the emperor.  As Marcus' devious son Commodus ascends to the throne, Maximus is set to be executed.  He escapes, but is captured by slave traders.  Renamed Spaniard and forced to become a gladiator, Maximus must battle to the death with other men for the amusement of paying audiences."
  },
  {
    title: "Piper",
    description: "A mother bird tries to teach her little one how to find food by herself. In the process, she encounters a traumatic experience that she must overcome in order to survive."
  },
  {
    title: "The Prestige",
    description: "A mysterious story of two magicians whose intense rivalry leads them on a life-long battle for supremacy -- full of obsession, deceit and jealousy with dangerous and deadly consequences."
  },
  {
    title: "Saving Private Ryan",
    description: "As U.S. troops storm the beaches of Normandy, three brothers lie dead on the battlefield, with a fourth trapped behind enemy lines. Ranger captain John Miller and seven men are tasked with penetrating German-held territory and bringing the boy home."
  },
  {
    title: "Memento",
    description: "Leonard Shelby is tracking down the man who raped and murdered his wife. The difficulty of locating his wife's killer, however, is compounded by the fact that he suffers from a rare, untreatable form of short-term memory loss. Although he can recall details of life before his accident, Leonard cannot remember what happened fifteen minutes ago, where he's going, or why."
  },
  {
    title: "Shutter Island",
    description: "World War II soldier-turned-U.S. Marshal Teddy Daniels investigates the disappearance of a patient from a hospital for the criminally insane, but his efforts are compromised by his troubling visions and also by a mysterious doctor."
  },
  {
    title: "The Matrix",
    description: "Set in the 22nd century, The Matrix tells the story of a computer hacker who joins a group of underground insurgents fighting the vast and powerful computers who now rule the earth."
  },
  {
    title: "My Mom is a Character 3",
    description: "Dona Herm\xEDnia will have to rediscover and reinvent herself because her children are forming new families. This supermom will have to deal with a new life scenario: Marcelina is pregnant and Juliano is getting married."
  },
  {
    title: "Hacksaw Ridge",
    description: "WWII American Army Medic Desmond T. Doss, who served during the Battle of Okinawa, refuses to kill people and becomes the first Conscientious Objector in American history to receive the Congressional Medal of Honor."
  },
  {
    title: "The Help",
    description: `Aibileen Clark is a middle-aged African-American maid who has spent her life raising white children and has recently lost her only son; Minny Jackson is an African-American maid who has often offended her employers despite her family's struggles with money and her desperate need for jobs; and Eugenia "Skeeter" Phelan is a young white woman who has recently moved back home after graduating college to find out her childhood maid has mysteriously disappeared. These three stories intertwine to explain how life in Jackson, Mississippi revolves around "the help"; yet they are always kept at a certain distance because of racial lines.`
  },
  {
    title: "Capernaum",
    description: "Zain, a 12-year-old boy scrambling to survive on the streets of Beirut, sues his parents for having brought him into such an unjust world, where being a refugee with no documents means that your rights can easily be denied."
  },
  {
    title: "Soul",
    description: "Joe Gardner is a middle school teacher with a love for jazz music. After a successful audition at the Half Note Club, he suddenly gets into an accident that separates his soul from his body and is transported to the You Seminar, a center in which souls develop and gain passions before being transported to a newborn child. Joe must enlist help from the other souls-in-training, like 22, a soul who has spent eons in the You Seminar, in order to get back to Earth."
  },
  {
    title: "All About Eve",
    description: "From the moment she glimpses her idol at the stage door, Eve Harrington is determined to take the reins of power away from the great actress Margo Channing. Eve maneuvers her way into Margo's Broadway role, becomes a sensation and even causes turmoil in the lives of Margo's director boyfriend, her playwright and his wife. Only the cynical drama critic sees through Eve, admiring her audacity and perfect pattern of deceit."
  },
  {
    title: "Taxi Driver",
    description: "A mentally unstable Vietnam War veteran works as a night-time taxi driver in New York City where the perceived decadence and sleaze feed his urge for violent action."
  },
  {
    title: "Wonder",
    description: "The story of August Pullman \u2013 a boy with facial differences \u2013 who enters fifth grade, attending a mainstream elementary school for the first time."
  },
  {
    title: "Singin' in the Rain",
    description: "In 1927 Hollywood, a silent film production company and cast make a difficult transition to sound."
  },
  {
    title: "Metropolis",
    description: "In a futuristic city sharply divided between the working class and the city planners, the son of the city's mastermind falls in love with a working class prophet who predicts the coming of a savior to mediate their differences."
  },
  {
    title: "Yojimbo",
    description: "A nameless ronin, or samurai with no master, enters a small village in feudal Japan where two rival businessmen are struggling for control of the local gambling trade. Taking the name Sanjuro Kuwabatake, the ronin convinces both silk merchant Tazaemon and sake merchant Tokuemon to hire him as a personal bodyguard, then artfully sets in motion a full-scale gang war between the two ambitious and unscrupulous men."
  },
  {
    title: "Joker",
    description: "During the 1980s, a failed stand-up comedian is driven insane and turns to a life of crime and chaos in Gotham City while becoming an infamous psychopathic crime figure."
  },
  {
    title: "Scenes from a Marriage",
    description: "Johan and Marianne are married and seem to have it all. Their happiness, however, is a fa\xE7ade for a troubled relationship, which becomes even rockier when Johan admits that he's having an affair. Before long, the spouses separate and move towards finalizing their divorce, but they make attempts at reconciling. Even as they pursue other relationships, Johan and Marianne realize that they have a significant bond, but also many issues that hinder that connection."
  },
  {
    title: "Casablanca",
    description: "In Casablanca, Morocco in December 1941, a cynical American expatriate meets a former lover, with unforeseen complications."
  },
  {
    title: "My Friends",
    description: 'Four middle-aged friends in Florence organize together idle pranks (called zingarate, "gypsy shenanigans") in a continuous attempt to prolong childhood during their adult life.'
  },
  {
    title: "La Dolce Vita",
    description: "Journalist and man-about-town Marcello struggles to find his place in the world, torn between the allure of Rome's elite social scene and the stifling domesticity offered by his girlfriend, all the while searching for a way to become a serious writer."
  },
  {
    title: "Me Against You: Mr. S's Vendetta",
    description: "A young couple who makes popular YouTube videos for children sets out to win an award, but an evil mastermind stands in the way of their success."
  },
  {
    title: "Bo Burnham: Make Happy",
    description: "Combining his trademark wit and self-deprecating humor with original music, Bo Burnham offers up his unique twist on life in this stand-up special about  life, death, sexuality, hypocrisy, mental illness and Pringles cans."
  },
  {
    title: "Andrei Rublev",
    description: "An expansive Russian drama, this film focuses on the life of revered religious icon painter Andrei Rublev. Drifting from place to place in a tumultuous era, the peace-seeking monk eventually gains a reputation for his art. But after Rublev witnesses a brutal battle and unintentionally becomes involved, he takes a vow of silence and spends time away from his work. As he begins to ease his troubled soul, he takes steps towards becoming a painter once again."
  },
  {
    title: "The Father",
    description: "A man refuses all assistance from his daughter as he ages and, as he tries to make sense of his changing circumstances, he begins to doubt his loved ones, his own mind and even the fabric of his reality."
  },
  {
    title: "Scarface",
    description: "After getting a green card in exchange for assassinating a Cuban government official, Tony Montana stakes a claim on the drug trade in Miami. Viciously murdering anyone who stands in his way, Tony eventually becomes the biggest drug lord in the state, controlling nearly all the cocaine that comes through Miami. But increased pressure from the police, wars with Colombian drug cartels and his own drug-fueled paranoia serve to fuel the flames of his eventual downfall."
  },
  {
    title: "The Departed",
    description: "To take down South Boston's Irish Mafia, the police send in one of their own to infiltrate the underworld, not realizing the syndicate has done likewise. While an undercover cop curries favor with the mob kingpin, a career criminal rises through the police ranks. But both sides soon discover there's a mole among them."
  },
  {
    title: "Rashomon",
    description: `Brimming with action while incisively examining the nature of truth, "Rashomon" is perhaps the finest film ever to investigate the philosophy of justice. Through an ingenious use of camera and flashbacks, Kurosawa reveals the complexities of human nature as four people recount different versions of the story of a man's murder and the rape of his wife.`
  },
  {
    title: "Lock, Stock and Two Smoking Barrels",
    description: "A card shark and his unwillingly-enlisted friends need to make a lot of cash quick after losing a sketchy poker match. To do this they decide to pull a heist on a small-time gang who happen to be operating out of the flat next door."
  },
  {
    title: "Togo",
    description: "The untold true story set in the winter of 1925 that takes you across the treacherous terrain of the Alaskan tundra for an exhilarating and uplifting adventure that will test the strength, courage and determination of one man, Leonhard Seppala, and his lead sled dog, Togo."
  },
  {
    title: "Ayla: The Daughter of War",
    description: "In 1950, amidst the ravages of the Korean War, Sergeant S\xFCleyman stumbles upon a a half-frozen little girl, with no parents and no help in sight. Frantic, scared and on the verge of death, this little girl captures the heart of S\xFCleyman, who risks his own life to save her, smuggling her into his Army base and out of harms way. Not knowing her name and unable to communicate with her, S\xFCleyman names her Ayla, in reminiscence of the moon on the fateful night during which they met. The two form an instantaneous and inseparable bond, and Ayla, almost effortlessly, brings an uncanny joy to the Turkish brigade in the grip of war. As the war comes to a close however, S\xFCleyman's brigade is told that they will be returning home. S\xFCleyman cannot bear abandoning Ayla, and does everything within his power to take her with him. After repeated failure, he is forced to give Ayla to an orphanage, but doesn't give up on the hope of one day reuniting with her. Will the two ever get back together?"
  },
  {
    title: "Double Indemnity",
    description: "A rich woman and a calculating insurance agent plot to kill her unsuspecting husband after he signs a double indemnity policy. Against a backdrop of distinctly Californian settings, the partners in crime plan the perfect murder to collect the insurance, which pays double if the death is accidental."
  },
  {
    title: "Central Station",
    description: "An emotive journey of a former school teacher, who writes letters for illiterate people, and a young boy, whose mother has just died, as they search for the father he never knew."
  },
  {
    title: "Sherlock Jr.",
    description: "A film projectionist longs to be a detective, and puts his meagre skills to work when he is framed by a rival for stealing his girlfriend's father's pocketwatch."
  },
  {
    title: "Nobody",
    description: `Hutch Mansell, a suburban dad, overlooked husband, nothing neighbor \u2014 a "nobody." When two thieves break into his home one night, Hutch's unknown long-simmering rage is ignited and propels him on a brutal path that will uncover dark secrets he fought to leave behind.`
  },
  {
    title: "Out of the Clear Blue Sky",
    description: "Returning to Earth as an imitator, the legendary Mexican artist Pedro Infante must prove that he is no longer a womanizer to enter paradise."
  },
  {
    title: "Reservoir Dogs",
    description: "A botched robbery indicates a police informant, and the pressure mounts in the aftermath at a warehouse. Crime begets violence as the survivors -- veteran Mr. White, newcomer Mr. Orange, psychopathic parolee Mr. Blonde, bickering weasel Mr. Pink and Nice Guy Eddie -- unravel."
  },
  {
    title: "Django Unchained",
    description: "With the help of a German bounty hunter, a freed slave sets out to rescue his wife from a brutal Mississippi plantation owner."
  },
  {
    title: "Far From the Tree",
    description: "On an idyllic beach in the Pacific Northwest, curiosity gets the better of a young raccoon whose frustrated parent attempts to keep them both safe."
  },
  {
    title: "Cruella",
    description: "In 1970s London amidst the punk rock revolution, a young grifter named Estella is determined to make a name for herself with her designs. She befriends a pair of young thieves who appreciate her appetite for mischief, and together they are able to build a life for themselves on the London streets. One day, Estella\u2019s flair for fashion catches the eye of the Baroness von Hellman, a fashion legend who is devastatingly chic and terrifyingly haute. But their relationship sets in motion a course of events and revelations that will cause Estella to embrace her wicked side and become the raucous, fashionable and revenge-bent Cruella."
  },
  {
    title: "The Great War",
    description: "Italy, 1916. Oreste Jacovacci and Giovanni Busacca are called, as all the Italian youths, to serve the army in the WWI. They both try in every way to avoid serving the army."
  },
  {
    title: "Full Metal Jacket",
    description: "A pragmatic U.S. Marine observes the dehumanizing effects the U.S.-Vietnam War has on his fellow recruits from their brutal boot camp training to the bloody street fighting in Hue."
  },
  {
    title: "Good Will Hunting",
    description: "Will Hunting has a genius-level IQ but chooses to work as a janitor at MIT. When he solves a difficult graduate-level math problem, his talents are discovered by Professor Gerald Lambeau, who decides to help the misguided youth reach his potential. When Will is arrested for attacking a police officer, Professor Lambeau makes a deal to get leniency for him if he will get treatment from therapist Sean Maguire."
  },
  {
    title: "Some Like It Hot",
    description: "Two musicians witness a mob hit and struggle to find a way out of the city before they are found by the gangsters. Their only opportunity is to join an all-girl band as they leave on a tour. To make their getaway they must first disguise themselves as women, then keep their identities secret and deal with the problems this brings - such as an attractive bandmate and a very determined suitor."
  },
  {
    title: "Dr. Strangelove or: How I Learned to Stop Worrying and Love the Bomb",
    description: "After the insane General Jack D. Ripper initiates a nuclear strike on the Soviet Union, a war room full of politicians, generals and a Russian diplomat all frantically try to stop the nuclear strike."
  },
  {
    title: "Wild Strawberries",
    description: "Crotchety retired doctor Isak Borg travels from Stockholm to Lund, Sweden, with his pregnant and unhappy daughter-in-law, Marianne, in order to receive an honorary degree from his alma mater. Along the way, they encounter a series of hitchhikers, each of whom causes the elderly doctor to muse upon the pleasures and failures of his own life. These include the vivacious young Sara, a dead ringer for the doctor's own first love."
  },
  {
    title: "The Invisible Guest",
    description: "Barcelona, Spain. Adri\xE1n Doria, a young and successful businessman accused of murder, meets one night with Virginia Goodman, an expert interrogation lawyer, in order to devise a defense strategy."
  },
  {
    title: "Alien",
    description: "During its return to the earth, commercial spaceship Nostromo intercepts a distress signal from a distant planet. When a three-member team of the crew discovers a chamber containing thousands of eggs on the planet, a creature inside one of the eggs attacks an explorer. The entire crew is unaware of the impending nightmare set to descend upon them when the alien parasite planted inside its unfortunate host is birthed."
  },
  {
    title: "Paper Lives",
    description: "In the streets of Istanbul, ailing waste warehouse worker Mehmet takes a small boy under his wing and must soon confront his own traumatic childhood."
  },
  {
    title: "How to Train Your Dragon: Homecoming",
    description: "It's been ten years since the dragons moved to the Hidden World, and even though Toothless doesn't live in New Berk anymore, Hiccup continues the holiday traditions he once shared with his best friend. But the Vikings of New Berk were beginning to forget about their friendship with dragons. Hiccup, Astrid, and Gobber know just what to do to keep the dragons in the villagers' hearts. And across the sea, the dragons have a plan of their own..."
  },
  {
    title: "Paris, Texas",
    description: "A man wanders out of the desert not knowing who he is. His brother finds him, and helps to pull his memory back of the life he led before he walked out on his family and disappeared four years earlier."
  },
  {
    title: "Pather Panchali",
    description: "Impoverished priest Harihar Ray, dreaming of a better life for himself and his family, leaves his rural Bengal village in search of work."
  },
  {
    title: "The Tale of The Princess Kaguya",
    description: "Found inside a shining stalk of bamboo by an old bamboo cutter and his wife, a tiny girl grows rapidly into an exquisite young lady. The mysterious young princess enthrals all who encounter her. But, ultimately, she must confront her fate, the punishment for her crime."
  },
  {
    title: "Anne of Green Gables",
    description: "At the turn of the century on Prince Edward Island, Matthew Cuthbert and his sister Marilla decide to take on an orphan boy as help for their farm. But they get an unexpected jolt when they're mistakenly sent a girl instead: Anne Shirley."
  },
  {
    title: "Harry Potter and the Deathly Hallows: Part 2",
    description: "Harry, Ron and Hermione continue their quest to vanquish the evil Voldemort once and for all. Just as things begin to look hopeless for the young wizards, Harry discovers a trio of magical objects that endow him with powers to rival Voldemort's formidable skills."
  },
  {
    title: "Dersu Uzala",
    description: "A military explorer meets and befriends a Goldi man in Russia\u2019s unmapped forests. A deep and abiding bond evolves between the two men, one civilized in the usual sense, the other at home in the glacial Siberian woods."
  },
  {
    title: "Big Deal on Madonna Street",
    description: "Best friends Peppe and Mario are thieves, but they're not very good at it. Still, Peppe thinks that he's finally devised a master heist that will make them rich. With the help of some fellow criminals, he plans to dig a tunnel from a rented apartment to the pawnshop next door, where they can rob the safe. But his plan is far from foolproof, and the fact that no one in the group has any experience digging tunnels proves to be the least of their problems."
  },
  {
    title: "Michael Jackson's Thriller",
    description: "A night at the movies turns into a nightmare when Michael and his date are attacked by a horde of bloody-thirsty zombies."
  },
  {
    title: "The Truman Show",
    description: "Truman Burbank is the star of The Truman Show, a 24-hour-a-day reality TV show that broadcasts every aspect of his life without his knowledge. His entire life has been an unending soap opera for consumption by the rest of the world. And everyone he knows, including his wife and his best friend is really an actor, paid to be part of his life."
  },
  {
    title: "Red Beard",
    description: "Aspiring to an easy job as personal physician to a wealthy family, Noboru Yasumoto is disappointed when his first post after medical school takes him to a small country clinic under the gruff doctor Red Beard. Yasumoto rebels in numerous ways, but Red Beard proves a wise and patient teacher. He gradually introduces his student to the unglamorous side of the profession, ultimately assigning him to care for a prostitute rescued from a local brothel."
  },
  {
    title: "M",
    description: "In this classic German thriller, Hans Beckert, a serial killer who preys on children, becomes the focus of a massive Berlin police manhunt. Beckert's heinous crimes are so repellant and disruptive to city life that he is even targeted by others in the seedy underworld network. With both cops and criminals in pursuit, the murderer soon realizes that people are on his trail, sending him into a tense, panicked attempt to escape justice."
  },
  {
    title: "Believe Me: The Abduction of Lisa McVey",
    description: "On the night she plans on taking her own life, 17-year-old 'Lisa McVey' is kidnapped and finds herself fighting to stay alive and manages to be a victim of rape. She manages to talk her attacker into releasing her, but when she returns home, no one believes her story except for one detective, who suspects she was abducted by a serial killer. Based on horrifying true events."
  },
  {
    title: "In the Mood for Love",
    description: "Hong Kong, 1962: Chow Mo-Wan and Su Li-Zhen move into neighboring apartments on the same day. Their encounters are formal and polite\u2014until a discovery about their spouses creates an intimate bond between them."
  },
  {
    title: "Incendies",
    description: "A mother's last wishes send twins Jeanne and Simon on a journey to Middle East in search of their tangled roots. Adapted from Wajdi Mouawad's acclaimed play, Incendies tells the powerful and moving tale of two young adults' voyage to the core of deep-rooted hatred, never-ending wars and enduring love."
  },
  {
    title: "A Special Day",
    description: "In Rome, fascist supporter Emanuele attends a parade commemorating Adolf Hitler's historic meeting with Italian leader Benito Mussolini, leaving his apolitical wife, Antonietta, to tend to household duties. Antonietta encounters a man, Gabriele, who appears surprisingly nonplussed by the political event. Over the course of the day, the two forge a close friendship that will forever change their perceptions of life, love and politics."
  },
  {
    title: "I corti",
    description: 'I corti ("shorts") by Aldo, Giovanni & Giacomo was the first stage show of the comedy trio, with the participation of Marina Massironi. It was recorded live at the Teatro Nuovo in Ferrara on 28 and 29 March 1996. Produced by Agidi, with the theatre direction of Arturo Brachetti.'
  },
  {
    title: "Hannah Gadsby: Nanette",
    description: "The Australian comedian Hannah Gadsby is taking an anti-comedy stance in her newest special."
  },
  {
    title: "The Hunt",
    description: "A teacher lives a lonely life, all the while struggling over his son\u2019s custody. His life slowly gets better as he finds love and receives good news from his son, but his new luck is about to be brutally shattered by an innocent little lie."
  },
  {
    title: "There Will Be Blood",
    description: "Ruthless silver miner, turned oil prospector, Daniel Plainview, moves to oil-rich California. Using his son to project a trustworthy, family-man image, Plainview cons local landowners into selling him their valuable properties for a pittance. However, local preacher Eli Sunday suspects Plainview's motives and intentions, starting a slow-burning feud that threatens both their lives."
  },
  {
    title: "Eternal Sunshine of the Spotless Mind",
    description: "Joel Barish, heartbroken that his girlfriend underwent a procedure to erase him from her memory, decides to do the same. However, as he watches his memories of her fade away, he realises that he still loves her, and may be too late to correct his mistake."
  },
  {
    title: "Three Billboards Outside Ebbing, Missouri",
    description: "After seven months have passed without a culprit in her daughter's murder case, Mildred Hayes makes a bold move, painting three signs leading into her town with a controversial message directed at Bill Willoughby, the town's revered chief of police. When his second-in-command Officer Jason Dixon, an immature mother's boy with a penchant for violence, gets involved, the battle between Mildred and Ebbing's law enforcement is only exacerbated."
  },
  {
    title: "Lion",
    description: "A five-year-old Indian boy gets lost on the streets of Calcutta, thousands of kilometers from home. He survives many challenges before being adopted by a couple in Australia; 25 years later, he sets out to find his lost family."
  },
  {
    title: "In the Arms of an Assassin",
    description: "Victor (William Levy) is one of the world\u2019s most handsome  men, but he has a deep secret \u2013 he is a cold blooded assassin. Smooth talking and seductive, Victor was raised to do one thing only, which is to kill for money. When he is sent to the home of a brutal drug lord to collect payment for his most recent hit, he encounters the beautiful Sarai (Alicia Sanz), who has been forced to spend the last 9 years of her life with the drug lord."
  },
  {
    title: "Straight Outta Nowhere: Scooby-Doo! Meets Courage the Cowardly Dog",
    description: "With Mystery, Inc. on the tail of a strange object in Nowhere, Kansas, the strange hometown of Eustice, Muriel, and Courage, the gang soon find themselves contending with a giant cicada monster and her winged warriors."
  },
  {
    title: "Kitbull",
    description: "An unlikely connection sparks between two creatures: a fiercely independent stray kitten and a pit bull. Together, they experience friendship for the first time."
  },
  {
    title: "Ugetsu",
    description: "In 16th century Japan, peasants Genjuro and Tobei sell their earthenware pots to a group of soldiers in a nearby village, in defiance of a local sage's warning against seeking to profit from warfare. Genjuro's pursuit of both riches and the mysterious Lady Wakasa, as well as Tobei's desire to become a samurai, run the risk of destroying both themselves and their wives, Miyagi and Ohama."
  },
  {
    title: "The Best of Youth",
    description: "Spanning nearly four decades, this generational epic follows two Italian brothers from a middle-class family through some of the most significant events of postwar Italian history after their life paths diverge thanks to one fateful encounter during the summer of 1966."
  },
  {
    title: "Toto, Peppino, and the Hussy",
    description: `Antonio, Peppino and Lucia are three brothers who live in the country near Naples. Lucia's son, Gianni, goes to Naples to study medicine, but there he knows a ballet dancer. They fall in love and, when she goes to Milan, Gianni follows her. Informed of this and afraid that their nephew will stop studying, the three Caponi brothers leave for Milan to persuade Gianni to come back and continue studying and abandon the "Malafemmina" (bad girl).`
  },
  {
    title: "The Passion of Joan of Arc",
    description: "A classic of the silent age, this film tells the story of the doomed but ultimately canonized 15th-century teenage warrior. On trial for claiming she'd spoken to God, Jeanne d'Arc is subjected to inhumane treatment and scare tactics at the hands of church court officials. Initially bullied into changing her story, Jeanne eventually opts for what she sees as the truth. Her punishment, a famously brutal execution, earns her perpetual martyrdom."
  },
  {
    title: "The 400 Blows",
    description: "For young Parisian boy Antoine Doinel, life is one difficult situation after another. Surrounded by inconsiderate adults, including his neglectful parents, Antoine spends his days with his best friend, Rene, trying to plan for a better life. When one of their schemes goes awry, Antoine ends up in trouble with the law, leading to even more conflicts with unsympathetic authority figures."
  },
  {
    title: "Scooby-Doo! and Kiss: Rock and Roll Mystery",
    description: "Get ready to Rock! Scooby-Doo and the Mystery Inc. Gang team up with the one and only KISS in this all-new, out-of-this-world adventure! We join the Gang at Kiss World \u2013 the all-things-Kiss theme park, as they investigate a series of strange hauntings. With help from Kiss, they discover that the Crimson Witch has returned to summon The Destroyer from the alternate dimension of Kissteria! The evil duos ghastly plan, to destroy the earth! Can the Gang's cunning and Kiss's power of rock save the day?!"
  },
  {
    title: "La Haine",
    description: "After a chaotic night of rioting in a marginal suburb of Paris, three young friends, Vinz, Hubert and Sa\xEFd, wander around unoccupied waiting for news about the state of health of a mutual friend who has been seriously injured when confronting the police."
  },
  {
    title: "Das Boot",
    description: "A German submarine hunts allied ships during the Second World War, but it soon becomes the hunted. The crew tries to survive below the surface, while stretching both the boat and themselves to their limits."
  },
  {
    title: "Love, Simon",
    description: "Everyone deserves a great love story. But for seventeen-year old Simon Spier it's a little more complicated: he's yet to tell his family or friends he's gay and he doesn't know the identity of the anonymous classmate he's fallen for online."
  },
  {
    title: "Pride & Prejudice",
    description: "A story of love and life among the landed English gentry during the Georgian era. Mr. Bennet is a gentleman living in Hertfordshire with his overbearing wife and five daughters, but if he dies their house will be inherited by a distant cousin whom they have never met, so the family's future happiness and security is dependent on the daughters making good marriages."
  },
  {
    title: "The Elephant Man",
    description: 'A Victorian surgeon rescues a heavily disfigured man being mistreated by his "owner" as a side-show freak. Behind his monstrous fa\xE7ade, there is revealed a person of great intelligence and sensitivity. Based on the true story of Joseph Merrick (called John Merrick in the film), a severely deformed man in 19th century London.'
  },
  {
    title: "A Taxi Driver",
    description: "May, 1980. Man-seob is a taxi driver in Seoul who lives from hand to mouth, raising his young daughter alone. One day, he hears that there is a foreigner who will pay big money for a drive down to Gwangju city. Not knowing that he\u2019s a German journalist with a hidden agenda, Man-seob takes the job."
  },
  {
    title: "Silenced",
    description: "Based on actual events that took place at Gwangju Inhwa School for the hearing-impaired, where young deaf students were the victims of repeated sexual assaults by faculty members over a period of five years in the early 2000s."
  },
  {
    title: "Ran",
    description: "With Ran, legendary director Akira Kurosawa reimagines Shakespeare's King Lear as a singular historical epic set in sixteenth-century Japan. Majestic in scope, the film is Kurosawa's late-life masterpiece, a profound examination of the folly of war and the crumbling of one family under the weight of betrayal, greed, and the insatiable thirst for power."
  },
  {
    title: "Mirror",
    description: "A dying man in his forties recalls his childhood, his mother, the war and personal moments that tell of and juxtapose pivotal moments in Soviet history with daily life."
  },
  {
    title: "2001: A Space Odyssey",
    description: "Humanity finds a mysterious object buried beneath the lunar surface and sets off to find its origins with the help of HAL 9000, the world's most advanced super computer."
  },
  {
    title: "Jojo Rabbit",
    description: "A World War II satire that follows a lonely German boy whose world view is turned upside down when he discovers his single mother is hiding a young Jewish girl in their attic. Aided only by his idiotic imaginary friend, Adolf Hitler, Jojo must confront his blind nationalism."
  },
  {
    title: "Terminator 2: Judgment Day",
    description: "Nearly 10 years have passed since Sarah Connor was targeted for termination by a cyborg from the future. Now her son, John, the future leader of the resistance, is the target for a newer, more deadly terminator. Once again, the resistance has managed to send a protector back to attempt to save John and his mother Sarah."
  },
  {
    title: "Bound by Honor",
    description: 'Based on the true life experiences of poet Jimmy Santiago Baca, the film focuses on half-brothers Paco and Cruz, and their bi-racial cousin Miklo. It opens in 1972, as the three are members of an East L.A. gang known as the "Vatos Locos", and the story focuses on how a violent crime and the influence of narcotics alter their lives. Miklo is incarcerated and sent to San Quentin, where he makes a "home" for himself. Cruz becomes an exceptional artist, but a heroin addiction overcomes him with tragic results. Paco becomes a cop and an enemy to his "carnal", Miklo.'
  },
  {
    title: "Doctor Who: The Time of the Doctor",
    description: "Orbiting a quiet backwater planet, the massed forces of the universe's deadliest species gather, drawn to a mysterious message that echoes out to the stars. And amongst them, the Doctor. Rescuing Clara from a family Christmas dinner, the Time Lord and his best friend must learn what this enigmatic signal means for his own fate and that of the universe."
  },
  {
    title: "My Neighbor Totoro",
    description: "Two sisters move to the country with their father in order to be closer to their hospitalized mother, and discover the surrounding trees are inhabited by Totoros, magical spirits of the forest. When the youngest runs away from home, the older sister seeks help from the spirits to find her."
  },
  {
    title: "The Red Shoes",
    description: "In this classic drama, Vicky Page is an aspiring ballerina torn between her dedication to dance and her desire to love. While her imperious instructor, Boris Lermontov, urges to her to forget anything but ballet, Vicky begins to fall for the charming young composer Julian Craster. Eventually Vicky, under great emotional stress, must choose to pursue either her art or her romance, a decision that carries serious consequences."
  },
  {
    title: "Love Exposure",
    description: `The story of a teenage boy named Yu, who falls for Yoko, a girl he runs into while working as an "up-skirt" photographer in an offshoot of the porn industry. His attempts to woo her are complicated by a spot of cross-dressing - which convinces Yoko that she is lesbian - dalliances with kung-fu and crime, and a constant struggle with the guilt that's a legacy of his Catholic upbringing.`
  },
  {
    title: "Late Spring",
    description: "Noriko is perfectly happy living at home with her widowed father, Shukichi, and has no plans to marry -- that is, until her aunt Masa convinces Shukichi that unless he marries off his 27-year-old daughter soon, she will likely remain alone for the rest of her life. When Noriko resists Masa's matchmaking, Shukichi is forced to deceive his daughter and sacrifice his own happiness to do what he believes is right."
  },
  {
    title: "Nights of Cabiria",
    description: "Rome, 1957. A woman, Cabiria, is robbed and left to drown by her boyfriend, Giorgio. Rescued, she resumes her life and tries her best to find happiness in a cynical world. Even when she thinks her struggles are over and she has found happiness and contentment, things may not be what they seem."
  },
  {
    title: "Rocco and His Brothers",
    description: "When a widow\u2019s family moves to the big city, two of her sons become romantic rivals with deadly results."
  },
  {
    title: "Cool Kids Don't Cry",
    description: "Adaptation of one of Benelux most famous children's novels. Tough prime school girl Akkie loves soccer and can be a real bully. Love is the only thing she's scared of. When Akkie is diagnosed with Leukemia, she has to fight for her life. On the verge of going to high school, Akkie has to allow love to enter her life, and thus gain courage to accept the inevitable."
  },
  {
    title: "Memories of Murder",
    description: "1986 Gyunggi Province. The body of a young woman is found brutally raped and murdered. Two months later, a series of rapes and murders commences under similar circumstances. And in a country that had never known such crimes, the dark whispers about a serial murderer grow louder. A special task force is set up in the area, with two local detectives Park Doo-Man and Jo Young-Goo joined by a detective from Seoul who requested to be assigned to the case."
  },
  {
    title: "A Dog's Journey",
    description: "A dog finds the meaning of his own existence through the lives of the humans he meets."
  },
  {
    title: "Loving Vincent",
    description: "A young man comes to the last hometown of painter Vincent van Gogh to deliver the troubled artist's final letter and ends up investigating his final days there."
  },
  {
    title: "Ron's Gone Wrong",
    description: "In a world where walking, talking, digitally connected bots have become children's best friends, an 11-year-old finds that his robot buddy doesn't quite work the same as the others do."
  },
  {
    title: "WALL\xB7E",
    description: "WALL\xB7E is the last robot left on an Earth that has been overrun with garbage and all humans have fled to outer space. For 700 years he has continued to try and clean up the mess, but has developed some rather interesting human-like qualities. When a ship arrives with a sleek new type of robot, WALL\xB7E thinks he's finally found a friend and stows away on the ship when it leaves."
  },
  {
    title: "Prisoners",
    description: "Keller Dover faces a parent's worst nightmare when his 6-year-old daughter, Anna, and her friend go missing. The only lead is an old motorhome that had been parked on their street. The head of the investigation, Detective Loki, arrests the driver, but a lack of evidence forces Loki to release his only suspect. Dover, knowing that his daughter's life is at stake, decides that he has no choice but to take matters into his own hands."
  },
  {
    title: "Hidden Figures",
    description: "The untold story of Katherine G. Johnson, Dorothy Vaughan and Mary Jackson \u2013 brilliant African-American women working at NASA and serving as the brains behind one of the greatest operations in history \u2013 the launch of astronaut John Glenn into orbit. The visionary trio crossed all gender and race lines to inspire generations to dream big."
  },
  {
    title: "Elite Squad",
    description: "In 1997, before the visit of the pope to Rio de Janeiro, Captain Nascimento from BOPE (Special Police Operations Battalion) is assigned to eliminate the risks of the drug dealers in a dangerous slum nearby where the pope intends to be lodged."
  },
  {
    title: "My Name Is Khan",
    description: "Rizwan Khan, a Muslim from the Borivali section of Mumbai, has Asperger's syndrome. He marries a Hindu single mother, Mandira, in San Francisco. After 9/11, Rizwan is detained by authorities at LAX who treat him as a terrorist because of his condition and his race."
  },
  {
    title: "Judgment at Nuremberg",
    description: "In 1947, four German judges who served on the bench during the Nazi regime face a military tribunal to answer charges of crimes against humanity. Chief Justice Haywood hears evidence and testimony not only from lead defendant Ernst Janning and his defense attorney Hans Rolfe, but also from the widow of a Nazi general, an idealistic U.S. Army captain and reluctant witness Irene Wallner."
  },
  {
    title: "Weathering with You",
    description: "The summer of his high school freshman year, Hodaka runs away from his remote island home to Tokyo, and quickly finds himself pushed to his financial and personal limits. The weather is unusually gloomy and rainy every day, as if taking its cue from his life. After many days of solitude, he finally finds work as a freelance writer for a mysterious occult magazine. Then, one day, Hodaka meets Hina on a busy street corner. This bright and strong-willed girl possesses a strange and wonderful ability: the power to stop the rain and clear the sky."
  },
  {
    title: "On My Skin",
    description: "The incredible true story behind the most controversial Italian court cases in recent years. Stefano Cucchi was arrested for a minor crime and mysteriously found dead during his detention. In one week's time, a family is changed forever."
  },
  {
    title: "Sing 2",
    description: "Buster and his new cast now have their sights set on debuting a new show at the Crystal Tower Theater in glamorous Redshore City. But with no connections, he and his singers must sneak into the Crystal Entertainment offices, run by the ruthless wolf mogul Jimmy Crystal, where the gang pitches the ridiculous idea of casting the lion rock legend Clay Calloway in their show. Buster must embark on a quest to find the now-isolated Clay and persuade him to return to the stage."
  },
  {
    title: "Room",
    description: "Held captive for 7 years in an enclosed space, a woman and her young son finally gain their freedom, allowing the boy to experience the outside world for the first time."
  },
  {
    title: "The Grand Budapest Hotel",
    description: "The Grand Budapest Hotel tells of a legendary concierge at a famous European hotel between the wars and his friendship with a young employee who becomes his trusted prot\xE9g\xE9. The story involves the theft and recovery of a priceless Renaissance painting, the battle for an enormous family fortune and the slow and then sudden upheavals that transformed Europe during the first half of the 20th century."
  },
  {
    title: "Finch",
    description: "On a post-apocalyptic Earth, a robot, built to protect the life of his dying creator's beloved dog, learns about life, love, friendship, and what it means to be human."
  },
  {
    title: "Spider-Man: No Way Home",
    description: "Peter Parker is unmasked and no longer able to separate his normal life from the high-stakes of being a super-hero. When he asks for help from Doctor Strange the stakes become even more dangerous, forcing him to discover what it truly means to be Spider-Man."
  },
  {
    title: "Gifted",
    description: "Frank, a single man raising his child prodigy niece Mary, is drawn into a custody battle with his mother."
  },
  {
    title: "In a Heartbeat",
    description: "A closeted boy runs the risk of being outed by his own heart after it pops out of his chest to chase down the boy of his dreams."
  },
  {
    title: "Along with the Gods: The Two Worlds",
    description: "Having died unexpectedly, firefighter Ja-hong is taken to the afterlife by 3 afterlife guardians. Only when he passes 7 trials over 49 days and proves he was innocent in human life, he\u2019s able to reincarnate, and his 3 afterlife guardians are by his side to defend him in trial."
  },
  {
    title: "Poverty and Nobility",
    description: "Eugene, a young nobleman, asks two penniless idlers to impersonate his noble relatives to help him marry Gemma, the daughter of an enriched cook."
  },
  {
    title: "The Sting",
    description: "Set in the 1930s this intricate caper deals with an ambitious small-time crook and a veteran con man who seek revenge on a vicious crime lord who murdered one of their gang."
  },
  {
    title: "The Lives of Others",
    description: "A tragic love story set in East Berlin with the backdrop of an undercover Stasi controlled culture. Stasi captain Wieler is ordered to follow author Dreyman and plunges deeper and deeper into his life until he reaches the threshold of doubting the system."
  },
  {
    title: "Wish Dragon",
    description: "Determined teen Din is longing to reconnect with his childhood best friend when he meets a wish-granting dragon who shows him the magic of possibilities."
  },
  {
    title: "Chungking Express",
    description: "Two melancholic Hong Kong policemen fall in love: one with a mysterious underworld figure, the other with a beautiful and ethereal server at a late-night restaurant he frequents."
  },
  {
    title: "To Kill a Mockingbird",
    description: "Scout Finch, 6, and her older brother Jem live in sleepy Maycomb, Alabama, spending much of their time with their friend Dill and spying on their reclusive and mysterious neighbor, Boo Radley. When Atticus, their widowed father and a respected lawyer, defends a black man named Tom Robinson against fabricated rape charges, the trial and tangent events expose the children to evils of racism and stereotyping."
  },
  {
    title: "The Wages of Fear",
    description: "In the South American jungle, supplies of nitroglycerine are needed at a remote oil field. The oil company pays four men to deliver the supplies in two trucks. A tense rivalry develops between the two sets of drivers on the rough remote roads where the slightest jolt can result in death."
  },
  {
    title: "All My Life",
    description: "It was a chance meeting started by one of Sol\u2019s friends trying to chat up Jennifer. However, in the end, it was those two who hit it off. Sol enjoyed Jen\u2019s smile, her effort, and how silly she could be. Jen enjoyed Sol\u2019s cooking, his athleticism, and that he would join her in fun moments. As you can imagine, love bloomed, and things got serious. Jen\u2019s investment in Sol led to her pushing him to follow his dreams and even move in to save money. Sol\u2019s investment in Jen well, it led to him proposing. But what started as a liver tumor grew into full-on cancer, so with a diagnosis of 6 months to live, Sol and Jennifer try to make the best of it."
  },
  {
    title: "Innocent Voices",
    description: "A young boy, in an effort to have a normal childhood in 1980s El Salvador, is caught up in a dramatic fight for his life as he desperately tries to avoid the war which is raging all around him"
  },
  {
    title: "For a Few Dollars More",
    description: 'Two bounty hunters are in pursuit of "El Indio," one of the most wanted fugitives in the western territories, and his gang.'
  },
  {
    title: "The General",
    description: "During America\u2019s Civil War, Union spies steal engineer Johnnie Gray's beloved locomotive, 'The General'\u2014with Johnnie's lady love aboard an attached boxcar\u2014and he single-handedly must do all in his power to both get The General back and to rescue Annabelle."
  },
  {
    title: "The Second Mother",
    description: "After leaving her daughter Jessica in a small town in Pernambuco to be raised by relatives, Val spends the next 13 years working as a nanny to Fabinho in S\xE3o Paulo. She has financial stability but has to live with the guilt of having not raised Jessica herself. As Fabinho\u2019s university entrance exams approach, Jessica reappears in her life and seems to want to give her mother a second chance. However, Jessica has not been raised to be a servant and her very existence will turn Val\u2019s routine on its head. With precision and humour, the subtle and powerful forces that keep rigid class structures in place and how the youth may just be the ones to shake it all up."
  },
  {
    title: "Prayers for Bobby",
    description: "Bobby Griffith was his mother's favorite son, the perfect all-American boy growing up under deeply religious influences in Walnut Creek, California. Bobby was also gay. Struggling with a conflict no one knew of, much less understood, Bobby finally came out to his family."
  },
  {
    title: "Song of the Sea",
    description: "The story of the last Seal Child\u2019s journey home. After their mother\u2019s disappearance, Ben and Saoirse are sent to live with Granny in the city. When they resolve to return to their home by the sea, their journey becomes a race against time as they are drawn into a world Ben knows only from his mother\u2019s folktales. But this is no bedtime story; these fairy folk have been in our world far too long. It soon becomes clear to Ben that Saoirse is the key to their survival."
  },
  {
    title: "The Circus",
    description: "Charlie, a wandering tramp, becomes a circus handyman - soon the star of the show - and falls in love with the circus owner's stepdaughter."
  },
  {
    title: "In This Corner of the World",
    description: "Japan, 1943, during World War II. Young Suzu leaves her village near Hiroshima to marry and live with her in-laws in Kure, a military harbor. Her creativity to overcome deprivation quickly makes her indispensable at home. Inhabited by an ancestral wisdom, Suzu impregnates the simple gestures of everyday life with poetry and beauty. The many hardships, the loss of loved ones, the frequent air raids of the enemy, nothing alters her enthusiasm\u2026"
  },
  {
    title: "The Young and the Damned",
    description: "A group of juvenile delinquents lives a criminal, violent life in the festering slums of Mexico City, among them the young Pedro, whose morality is gradually corrupted and destroyed by the others."
  },
  {
    title: "The Treasure of the Sierra Madre",
    description: "Fred C. Dobbs and Bob Curtin, both down on their luck in Tampico, Mexico in 1925, meet up with a grizzled prospector named Howard and decide to join with him in search of gold in the wilds of central Mexico. Through enormous difficulties, they eventually succeed in finding gold, but bandits, the elements, and most especially greed threaten to turn their success into disaster."
  },
  {
    title: "Dial M for Murder",
    description: "An ex-tennis pro carries out a plot to have his wife murdered after discovering she is having an affair, and assumes she will soon leave him for the other man anyway. When things go wrong, he improvises a new plan\u2014to frame her for murder instead."
  },
  {
    title: "Palmer",
    description: "After 12 years in prison, former high school football star Eddie Palmer returns home to put his life back together\u2014and forms an unlikely bond with Sam, an outcast boy from a troubled home. But Eddie's past threatens to ruin his new life and family."
  },
  {
    title: "Bingo: The King of the Mornings",
    description: '1980s. Brazilian television exploding in color and auditorium programs not so politically correct. In the middle of this fervor, Augusto Mendes, a young rising actor, seeks his place in the sun. From porn studios to soap operas, he finally finds success and fame when he becomes "Bingo", a TV host clown from one of the audience leader TV shows for children. It turns out that behind the rice powder and red nose, nobody knows who he is.'
  },
  {
    title: "The Hidden Fortress",
    description: "Japanese peasants Matashichi and Tahei try and fail to make a profit from a tribal war. They find a man and woman whom they believe are simple tribe members hiding in a fortress. Although the peasants don't know that Rokurota is a general and Yuki is a princess, the peasants agree to accompany the pair to safety in return for gold. Along the way, the general must prove his expertise in battle while also hiding his identity."
  },
  {
    title: "Veintea\xF1era, Divorciada y Fant\xE1stica",
    description: "Regina, our young protagonist, always dreamed of getting married. And she did it - but the dream lasted much less than she thought and now she has to face life in a very funny way as a divorcee."
  },
  {
    title: "CODA",
    description: "As a CODA (Child of Deaf Adults), Ruby is the only hearing person in her deaf family. When the family's fishing business is threatened, Ruby finds herself torn between pursuing her love of music and her fear of abandoning her parents."
  },
  {
    title: "Ivan Vasilyevich Changes His Profession",
    description: "A scientist builds a time machine and accidentally sends his apartment complex manager and a petty burglar to 16th century Moscow, while Tsar Ivan the Terrible travels to 1973."
  },
  {
    title: "The Gold Rush",
    description: "A lone prospector ventures into Alaska looking for gold. He soon gets mixed up with some burly characters, falls in love with the beautiful Georgia, and tries to win her heart with his singular charm."
  },
  {
    title: "Rome, Open City",
    description: "During the Nazi occupation of Rome in 1944, the leader of the Resistance is chased by the Nazis as he seeks refuge and a way to escape."
  },
  {
    title: "American Beauty",
    description: "Lester Burnham, a depressed suburban father in a mid-life crisis, decides to turn his hectic life around after developing an infatuation with his daughter's attractive friend."
  },
  {
    title: "The Cranes Are Flying",
    description: "Veronika and Boris come together in Moscow shortly before World War II. Walking along the river, they watch cranes fly overhead, and promise to rendezvous before Boris leaves to fight. Boris misses the meeting and is off to the front lines, while Veronika waits patiently, sending letters faithfully. After her house is bombed, Veronika moves in with Boris' family, into the company of a cousin with his own intentions."
  },
  {
    title: "Requiem for a Dream",
    description: "The hopes and dreams of four ambitious people are shattered when their drug addictions begin spiraling out of control. A look into addiction and how it overcomes the mind and body."
  },
  {
    title: "The Wolf of Wall Street",
    description: "A New York stockbroker refuses to cooperate in a large securities fraud case involving corruption on Wall Street, corporate banking world and mob infiltration. Based on Jordan Belfort's autobiography."
  },
  {
    title: "Citizen Kane",
    description: "Newspaper magnate, Charles Foster Kane is taken from his mother as a boy and made the ward of a rich industrialist. As a result, every well-meaning, tyrannical or self-destructive move he makes for the rest of his life appears in some way to be a reaction to that deeply wounding event."
  },
  {
    title: "El mesero",
    description: "A waiter pretends to be an important businessman in order to reach the upper class through his entrepreneurial dreams."
  },
  {
    title: "The Deer Hunter",
    description: "A group of working-class friends decide to enlist in the Army during the Vietnam War and finds it to be hellish chaos -- not the noble venture they imagined. Before they left, Steven married his pregnant girlfriend -- and Michael and Nick were in love with the same woman. But all three are different men upon their return."
  },
  {
    title: "Amadeus",
    description: "Wolfgang Amadeus Mozart is a remarkably talented young Viennese composer who unwittingly finds a fierce rival in the disciplined and determined Antonio Salieri. Resenting Mozart for both his hedonistic lifestyle and his undeniable talent, the highly religious Salieri is gradually consumed by his jealousy and becomes obsessed with Mozart's downfall, leading to a devious scheme that has dire consequences for both men."
  },
  {
    title: "A Man Escaped",
    description: "A captured French Resistance fighter during World War II engineers a daunting escape from prison."
  },
  {
    title: "Nobody Knows",
    description: "In a small Tokyo apartment, twelve-year-old Akira must care for his younger siblings after their mother leaves them and shows no sign of returning."
  },
  {
    title: "The Boy and the Beast",
    description: "Kyuta, a boy living in Shibuya, and Kumatetsu, a lonesome beast from Jutengai, an imaginary world. One day, Kyuta forays into the imaginary world and, as he's looking for his way back, meets Kumatetsu who becomes his spirit guide. That encounter leads them to many adventures."
  },
  {
    title: "Miracle in Cell No. 7",
    description: "A story about a mentally ill man wrongfully accused of murder and his relationship with his lovingly adorable 6 year old daughter."
  },
  {
    title: "Harry Potter and the Prisoner of Azkaban",
    description: "Year three at Hogwarts means new fun and challenges as Harry learns the delicate art of approaching a Hippogriff, transforming shape-shifting Boggarts into hilarity and even turning back time. But the term also brings danger: soul-sucking Dementors hover over the school, an ally of the accursed He-Who-Cannot-Be-Named lurks within the castle walls, and fearsome wizard Sirius Black escapes Azkaban. And Harry will confront them all."
  },
  {
    title: "No Manches Frida 2",
    description: "Ex-con Zequi and company come back for a new adventure."
  },
  {
    title: "Where Hands Touch",
    description: "Germany, 1944. Leyna, the 15-year old daughter of a white German mother and a black African father, meets Lutz, a compassionate member of the Hitler Youth whose father is a prominent Nazi soldier, and they form an unlikely connection in this quickly changing world."
  },
  {
    title: "Autumn Sonata",
    description: "After a seven-year absence, Charlotte Andergast travels to Sweden to reunite with her daughter Eva. The pair have a troubled relationship: Charlotte sacrificed the responsibilities of motherhood for a career as a classical pianist. Over an emotional night, the pair reopen the wounds of the past. Charlotte gets another shock when she finds out that her mentally impaired daughter, Helena, is out of the asylum and living with Eva."
  },
  {
    title: "The Thing",
    description: "Members of an American scientific research outpost in Antarctica find themselves battling a parasitic alien organism capable of perfectly imitating its victims. They soon discover that this task will be harder than they thought, as they don't know which members of the team have already been assimilated and their paranoia threatens to tear them apart."
  },
  {
    title: "Bohemian Rhapsody",
    description: "Singer Freddie Mercury, guitarist Brian May, drummer Roger Taylor and bass guitarist John Deacon take the music world by storm when they form the rock 'n' roll band Queen in 1970. Hit songs become instant classics. When Mercury's increasingly wild lifestyle starts to spiral out of control, Queen soon faces its greatest challenge yet \u2013 finding a way to keep the band together amid the success and excess."
  },
  {
    title: "On the Waterfront",
    description: "Terry Malloy dreams about being a prize fighter, while tending his pigeons and running errands at the docks for Johnny Friendly, the corrupt boss of the dockers union. Terry witnesses a murder by two of Johnny's thugs, and later meets the dead man's sister and feels responsible for his death. She introduces him to Father Barry, who tries to force him to provide information for the courts that will smash the dock racketeers."
  },
  {
    title: "The Imitation Game",
    description: "Based on the real life story of legendary cryptanalyst Alan Turing, the film portrays the nail-biting race against time by Turing and his brilliant team of code-breakers at Britain's top-secret Government Code and Cypher School at Bletchley Park, during the darkest days of World War II."
  },
  {
    title: "Casino",
    description: `In early-1970s Las Vegas, low-level mobster Sam "Ace" Rothstein gets tapped by his bosses to head the Tangiers Casino. At first, he's a great success in the job, but over the years, problems with his loose-cannon enforcer Nicky Santoro, his ex-hustler wife Ginger, her con-artist ex Lester Diamond and a handful of corrupt politicians put Sam in ever-increasing danger.`
  },
  {
    title: "Like Stars on Earth",
    description: "Ishaan Awasthi is an eight-year-old whose world is filled with wonders that no one else seems to appreciate. Colours, fish, dogs, and kites don't seem important to the adults, who are much more interested in things like homework, marks, and neatness. Ishaan cannot seem to get anything right in class; he is then sent to boarding school, where his life changes forever."
  },
  {
    title: "Belle",
    description: `Suzu is a 17-year-old high-school student living in a rural town with her father. Wounded by the loss of her mother at a young age, Suzu one day discovers the massive online world "U" and dives into this alternate reality as her avatar, Belle. Before long, all of U's eyes are fixed on Belle, when, suddenly, a mysterious, dragon-like figure appears before her.`
  },
  {
    title: "The Mitchells vs. the Machines",
    description: "A quirky, dysfunctional family's road trip is upended when they find themselves in the middle of the robot apocalypse and suddenly become humanity's unlikeliest last hope."
  },
  {
    title: "Hachi: A Dog's Tale",
    description: "A drama based on the true story of a college professor's bond with the abandoned dog he takes into his home."
  },
  {
    title: "Pink Floyd: The Wall",
    description: "A troubled rock star descends into madness in the midst of his physical and social isolation from everyone."
  },
  {
    title: "The Way He Looks",
    description: "Leonardo is a blind teenager dealing with an overprotective mother while trying to live a more independent life. To the disappointment of his best friend, Giovana, he plans to go on an exchange program abroad. When Gabriel, a new student in town, arrives at their classroom, new feelings blossom in Leonardo making him question his plans."
  },
  {
    title: "The Secret in Their Eyes",
    description: "A retired legal counselor writes a novel hoping to find closure for one of his past unresolved homicide cases and for his unreciprocated love with his superior - both of which still haunt him decades later."
  },
  {
    title: "Tae Guk Gi: The Brotherhood of War",
    description: "When two brothers are forced to fight in the Korean War, the elder decides to take the riskiest missions if it will help shield the younger from battle."
  },
  {
    title: "Forgotten",
    description: "Seoul, South Korea, 1997. When the young but extremely anxious student Jin-seok, his parents and his successful older brother Yoo-seok move to a new home, mysterious and frightening events begin to happen around them, unexplained events that threaten to ruin their seemingly happy lives. Unable to understand what is happening, Jin-seok wonders if he is losing his mind."
  },
  {
    title: "La Notte",
    description: "A day in the life of an unfaithful married couple and their steadily deteriorating relationship in Milan."
  },
  {
    title: "Gone with the Wind",
    description: `The spoiled daughter of a well-to-do plantation owner is forced to use every means at her disposal to claw her way out of poverty, following Maj. Gen. William Sherman's destructive "March to the Sea,\u201D during the American Civil War.`
  },
  {
    title: "A Whisker Away",
    description: 'Miyo "Muge" Sasaki is a peculiar second-year junior high student who has fallen in love with her classmate Kento Hinode. Muge resolutely pursues Kento every day, but he takes no notice of her. Nevertheless, while carrying a secret she can tell no one, Muge continues to pursue Kento. Muge discovers a magic mask that allows her to transform into a cat named Tar\u014D. The magic lets Muge get close to Kento, but eventually it may also make her unable to transform back to a human.'
  },
  {
    title: "Paperman",
    description: "An urban office worker finds that paper airplanes are instrumental in meeting a girl in ways he never expected."
  },
  {
    title: "Gran Torino",
    description: "Disgruntled Korean War veteran Walt Kowalski sets out to reform his neighbor, Thao Lor, a Hmong teenager who tried to steal Kowalski's prized possession: a 1972 Gran Torino."
  },
  {
    title: "Barry Lyndon",
    description: "An Irish rogue uses his cunning and wit to work his way up the social classes of 18th century England, transforming himself from the humble Redmond Barry into the noble Barry Lyndon."
  },
  {
    title: "Ford v Ferrari",
    description: "American car designer Carroll Shelby and the British-born driver Ken Miles work together to battle corporate interference, the laws of physics, and their own personal demons to build a revolutionary race car for Ford Motor Company and take on the dominating race cars of Enzo Ferrari at the 24 Hours of Le Mans in France in 1966."
  },
  {
    title: "Sanjuro",
    description: `Toshiro Mifune swaggers and snarls to brilliant comic effect in Kurosawa's tightly paced, beautifully composed "Sanjuro." In this companion piece and sequel to "Yojimbo," jaded samurai Sanjuro helps an idealistic group of young warriors weed out their clan's evil influences, and in the process turns their image of a proper samurai on its ear.`
  },
  {
    title: "When Marnie Was There",
    description: "Upon being sent to live with relatives in the countryside due to an illness, an emotionally distant adolescent girl becomes obsessed with an abandoned mansion and infatuated with a girl who lives there - a girl who may or may not be real."
  },
  {
    title: "North by Northwest",
    description: "Advertising man Roger Thornhill is mistaken for a spy, triggering a deadly cross-country chase."
  },
  {
    title: "Dangal",
    description: "Dangal is an extraordinary true story based on the life of Mahavir Singh and his two daughters, Geeta and Babita Phogat. The film traces the inspirational journey of a father who trains his daughters to become world class wrestlers."
  },
  {
    title: "My Way",
    description: "Jun-shik, who works for Tatsuo\u2019s grandfather\u2019s farm while Korea is colonized by Japan, dreams about participating in the Tokyo Olympics as a marathon runner. Tatsuo also aims to become a marathon runner, so the two become rivals. But the war breaks out and they both are forced to enlist in the army. Tatsuo becomes the head of defense in Jun-shik\u2019s unit and he devises a scheme but fails. Jun-shik and Tatsuo are held captive by the Soviets. They run away but soon are captured by the Germans and forced to separate. In 1944, they meet again on the shores of Normandy."
  },
  {
    title: "Scooby-Doo! and the Curse of the 13th Ghost",
    description: "Mystery Inc. withdraws from solving crimes after botching a case. When Vincent Van Ghoul contacts the gang about an unfinished investigation from Daphne, Shaggy and Scooby's past, the gang springs into action to finish the job that involves catching the 13th Ghost that escaped from the Chest of Demons and is still at large."
  },
  {
    title: "Rope",
    description: "Two men attempt to prove they committed the perfect crime by hosting a dinner party after strangling their former classmate to death."
  },
  {
    title: "Raging Bull",
    description: "The life of boxer Jake LaMotta, whose violence and temper that led him to the top in the ring destroyed his life outside of it."
  },
  {
    title: "The Third Man",
    description: `In postwar Vienna, Austria, Holly Martins, a writer of pulp Westerns, arrives penniless as a guest of his childhood chum Harry Lime, only to learn he has died. Martins develops a conspiracy theory after learning of a "third man" present at the time of Harry's death, running into interference from British officer Major Calloway, and falling head-over-heels for Harry's grief-stricken lover, Anna.`
  },
  {
    title: "Operation Y and Other Shurik's Adventures",
    description: `The film consists of three independent parts: "Workmate", "D\xE9j\xE0 vu" and "Operation Y". The plot follows the adventures of Shurik (alternative spelling \u2014 Shourick), the naive and nerdy Soviet student who often gets into ludicrous situations but always finds a way out very neatly.  "Operation Y and Shurik's Other Adventures" was a hit movie and became the leader of Soviet film distribution in 1965.`
  },
  {
    title: "The Exterminating Angel",
    description: "After a lavish dinner party, the guests find themselves mysteriously unable to leave the room."
  },
  {
    title: "Castaway on the Moon",
    description: "Mr. Kim is jobless, lost in debt and has been dumped by his girlfriend. He decides to end it all by jumping into the Han River - only to find himself washed up on a small, mid-river island. He soon abandons thoughts of suicide or rescue and begins a new life as a castaway. His antics catch the attention of a young woman whose apartment overlooks the river. Her discovery changes both their lives."
  },
  {
    title: "Limelight",
    description: "A fading music hall comedian tries to help a despondent ballet dancer learn to walk and to again feel confident about life."
  },
  {
    title: "It's Such a Beautiful Day",
    description: "Bill struggles to put together his shattered psyche, in this feature film version of Don Hertzfeldt's animated short film trilogy."
  },
  {
    title: "Inner Workings",
    description: "This is the story of the internal struggle between a man's Brain\u2014a pragmatic protector who calculates his every move, and his Heart\u2014a free-spirited adventurer who wants to let loose."
  },
  {
    title: "Cries and Whispers",
    description: "As Agnes slowly dies of cancer, her sisters are so deeply immersed in their own psychic pains that they can't offer her the support she needs. Maria is wracked with guilt at her husband's attempted suicide, caused by his discovery of her extramarital affair. The self-loathing, suicidal Karin seems to regard her sister with revulsion. Only Anna, the deeply religious maid who lost her young child, seems able to offer Agnes solace and empathy."
  },
  {
    title: "Raya and the Last Dragon",
    description: "Long ago, in the fantasy world of Kumandra, humans and dragons lived together in harmony. But when an evil force threatened the land, the dragons sacrificed themselves to save humanity. Now, 500 years later, that same evil has returned and it\u2019s up to a lone warrior, Raya, to track down the legendary last dragon to restore the fractured land and its divided people."
  },
  {
    title: "Remi, Nobody's Boy",
    description: "At the age of 10 years, young R\xE9mi is snatched from his adoptive mother and entrusted to the signor Vitalis, a mysterious itinerant musician. Has its hard sides - he will learn the harsh life of acrobat and sing to win his bread. Accompanied by the faithful dog capi and of the small monkey Joli-Coeur, his long trip through France, made for meetings, friendships and mutual assistance, leads him to the secret of its origins."
  },
  {
    title: "Batman: The Dark Knight Returns, Part 2",
    description: "Batman has stopped the reign of terror that The Mutants had cast upon his city.  Now an old foe wants a reunion and the government wants The Man of Steel to put a stop to Batman."
  },
  {
    title: "Words on Bathroom Walls",
    description: "Diagnosed with a mental illness halfway through his senior year of high school, a witty, introspective teen struggles to keep it a secret while falling in love with a brilliant classmate who inspires him to open his heart and not be defined by his condition."
  },
  {
    title: "An Egg Rescue",
    description: "Toto and his friends must rescue his egg children after they're taken away for a gourmet food event in Africa."
  },
  {
    title: "Bad Genius",
    description: "Lynn, a brilliant student, after helping her friends to get the grades they need, develops the idea of starting a much bigger exam-cheating business."
  },
  {
    title: "Army of Shadows",
    description: "Betrayed by an informant, Philippe Gerbier finds himself trapped in a torturous Nazi prison camp. Though Gerbier escapes to rejoin the Resistance in occupied Marseilles, France, and exacts his revenge on the informant, he must continue a quiet, seemingly endless battle against the Nazis in an atmosphere of tension, paranoia and distrust."
  },
  {
    title: "Lawrence of Arabia",
    description: "The story of British officer T.E. Lawrence's mission to aid the Arab tribes in their revolt against the Ottoman Empire during the First World War. Lawrence becomes a flamboyant, messianic figure in the cause of Arab unity but his psychological instability threatens to undermine his achievements."
  },
  {
    title: "Trainspotting",
    description: "Heroin addict Mark Renton stumbles through bad ideas and sobriety attempts with his unreliable friends -- Sick Boy, Begbie, Spud and Tommy. He also has an underage girlfriend, Diane, along for the ride. After cleaning up and moving from Edinburgh to London, Mark finds he can't escape the life he left behind when Begbie shows up at his front door on the lam, and a scheming Sick Boy follows."
  },
  {
    title: "1917",
    description: "At the height of the First World War, two young British soldiers must cross enemy territory and deliver a message that will stop a deadly attack on hundreds of soldiers."
  },
  {
    title: "3 Idiots",
    description: `Rascal. Joker. Dreamer. Genius... You've never met a college student quite like "Rancho." From the moment he arrives at India's most prestigious university, Rancho's outlandish schemes turn the campus upside down\u2014along with the lives of his two newfound best friends. Together, they make life miserable for "Virus," the school\u2019s uptight and heartless dean. But when Rancho catches the eye of the dean's sexy daughter, Virus sets his sights on flunking out the "3 idiots" once and for all.`
  },
  {
    title: "Girl in the Basement",
    description: "Sara is a teen girl who is looking forward to her 18th birthday to move away from her controlling father Don. But before she could even blow out the candles, Don imprisons her in the basement of their home."
  },
  {
    title: "Castle in the Sky",
    description: "A young boy and a girl with a magic crystal must race against pirates and foreign agents in a search for a legendary floating castle."
  },
  {
    title: "Scooby-Doo! Camp Scare",
    description: "Scooby and the gang experience outdoor fun as they go back to Fred's old summer camp."
  },
  {
    title: "Divorce Italian Style",
    description: "Ferdinando Cefal\xF9 is desperate to marry his cousin, Angela, but he is married to Rosalia and divorce is illegal in Italy. To get around the law, he tries to trick his wife into having an affair so he can catch her and murder her, as he knows he would be given a light sentence for killing an adulterous woman. He persuades a painter to lure his wife into an affair, but Rosalia proves to be more faithful than he expected."
  },
  {
    title: "Kill Bill: Vol. 1",
    description: "An assassin is shot by her ruthless employer, Bill, and other members of their assassination circle \u2013 but she lives to plot her vengeance."
  },
  {
    title: "Three Men and a Leg",
    description: "Friends Aldo, Giovanni, and Giacomo travel from north to south for Giacomo's wedding, carrying a precious item: the father of the bride, a tyrannical rich man who is both their boss and father-in-law (also Aldo and Giovanni married his daughters), has entrusted them with a wooden leg, the work of a famous artist."
  },
  {
    title: "A Trip to the Moon",
    description: "Professor Barbenfouillis and five of his colleagues from the Academy of Astronomy travel to the Moon aboard a rocket propelled by a giant cannon. Once on the lunar surface, the bold explorers face the many perils hidden in the caves of the mysterious planet."
  },
  {
    title: "La Strada",
    description: "When Gelsomina, a na\xEFve young woman, is purchased from her impoverished mother by brutish circus strongman Zampan\xF2 to be his wife and partner, she loyally endures her husband's coldness and abuse as they travel the Italian countryside performing together. Soon Zampan\xF2 must deal with his jealousy and conflicted feelings about Gelsomina when she finds a kindred spirit in Il Matto, the carefree circus fool, and contemplates leaving Zampan\xF2."
  },
  {
    title: "What Ever Happened to Baby Jane?",
    description: "Two aging film actresses live as virtual recluses in an old Hollywood mansion. Jane Hudson, a successful child star, cares for her crippled sister Blanche, whose career in later years eclipsed that of Jane. Now the two live together, their relationship affected by simmering subconscious thoughts of mutual envy, hate and revenge."
  },
  {
    title: "Three Colors: Red",
    description: "Valentine, a student model in Geneva, struggles with a possessive boyfriend and a troubled family. When she runs over a dog, she discovers that its owner, a retired judge, is illegally wiretapping and eavesdropping on his neighbors' phone calls. Although Valentine is outraged, she develops a strange bond with the judge \u2013 and as the two become closer, she finds herself caught in the middle of events that could change her life."
  },
  {
    title: "Raise the Red Lantern",
    description: "China in the 1920s. After her father's death, 19 year old Songlian is forced to marry the much older lord of a powerful family. With three wives already, each living in a separate house within the great castle, there is fierce competition for his attention and the privileges that are gained. This competition gets out of hand."
  },
  {
    title: "Toy Story",
    description: "Led by Woody, Andy's toys live happily in his room until Andy's birthday brings Buzz Lightyear onto the scene. Afraid of losing his place in Andy's heart, Woody plots against Buzz. But when circumstances separate Buzz and Woody from their owner, the duo eventually learns to put aside their differences."
  },
  {
    title: "The Battle of Algiers",
    description: "Tracing the struggle of the Algerian Front de Liberation Nationale to gain freedom from French colonial rule as seen through the eyes of Ali from his start as a petty thief to his rise to prominence in the organisation and capture by the French in 1957. The film traces the rebels' struggle and the increasingly extreme measures taken by the French government to quell the revolt."
  },
  {
    title: "Throne of Blood",
    description: `Returning to their lord's castle, samurai warriors Washizu and Miki are waylaid by a spirit who predicts their futures. When the first part of the spirit's prophecy comes true, Washizu's scheming wife, Asaji, presses him to speed up the rest of the spirit's prophecy by murdering his lord and usurping his place. Director Akira Kurosawa's resetting of William Shakespeare's "Macbeth" in feudal Japan is one of his most acclaimed films.`
  },
  {
    title: "Million Dollar Baby",
    description: "Despondent over a painful estrangement from his daughter, trainer Frankie Dunn isn't prepared for boxer Maggie Fitzgerald to enter his life. But Maggie's determined to go pro and to convince Dunn and his cohort to help her."
  },
  {
    title: "My Father's Violin",
    description: "Through their shared grief and connection to music, an orphaned girl bonds with her emotionally aloof, successful violinist uncle."
  },
  {
    title: "The Cabinet of Dr. Caligari",
    description: "Francis, a young man, recalls in his memory the horrible experiences he and his fianc\xE9e Jane recently went through. Francis and his friend Alan visit The Cabinet of Dr. Caligari, an exhibit where the mysterious doctor shows the somnambulist Cesare, and awakens him for some moments from his death-like sleep."
  },
  {
    title: "Before Sunrise",
    description: "On his way to Vienna, American Jesse meets C\xE9line, a student returning to Paris. After long conversations forge a surprising connection between them, Jesse convinces Celine to get off the train with him in Vienna. Since his flight to the U.S. departs the next morning and he has no money for lodging, they wander the city together, taking in the experiences of Vienna and each other."
  },
  {
    title: "My Hero Academia: Two Heroes",
    description: "All Might and Deku accept an invitation to go abroad to a floating and mobile manmade city, called 'I-Island', where they research quirks as well as hero supplemental items at the special 'I-Expo' convention that is currently being held on the island. During that time, suddenly, despite an iron wall of security surrounding the island, the system is breached by a villain, and the only ones able to stop him are the students of Class 1-A."
  },
  {
    title: "Kill Bill: The Whole Bloody Affair",
    description: "An assassin is shot and almost killed by her ruthless employer, Bill, and other members of their assassination circle \u2013 but she lives to plot her vengeance.  Kill Bill: The Whole Bloody Affair is a complete edit of the two-part martial arts action films Kill Bill: Vol. 1 and Kill Bill: Vol. 2. The film was originally scheduled to be released as one part. However, due to the film's over 4 hour running time, it was split into two parts."
  },
  {
    title: "The Cameraman",
    description: "A photographer takes up newsreel shooting to impress a secretary."
  },
  {
    title: "Children of Heaven",
    description: "Zohre's shoes are gone; her older brother Ali lost them. They are poor, there are no shoes for Zohre until they come up with an idea: they will share one pair of shoes, Ali's. School awaits."
  },
  {
    title: "The Tomorrow War",
    description: "The world is stunned when a group of time travelers arrive from the year 2051 to deliver an urgent message: Thirty years in the future, mankind is losing a global war against a deadly alien species. The only hope for survival is for soldiers and civilians from the present to be transported to the future and join the fight. Among those recruited is high school teacher and family man Dan Forester. Determined to save the world for his young daughter, Dan teams up with a brilliant scientist and his estranged father in a desperate quest to rewrite the fate of the planet."
  },
  {
    title: "Where Is My Friend's House?",
    description: "An 8 year old boy must return his friend's notebook he took by mistake, lest his friend be punished by expulsion from school."
  },
  {
    title: "Akira",
    description: "A secret military project endangers Neo-Tokyo when it turns a biker gang member into a rampaging psychic psychopath that only two teenagers and a group of psychics can stop."
  },
  {
    title: "Amarcord",
    description: "In an Italian seaside town, young Titta gets into trouble with his friends and watches various local eccentrics as they engage in often absurd behavior. Frequently clashing with his stern father and defended by his doting mother, Titta witnesses the actions of a wide range of characters, from his extended family to Fascist loyalists to sensual women, with certain moments shifting into fantastical scenarios."
  },
  {
    title: "Catch Me If You Can",
    description: "A true story about Frank Abagnale Jr. who, before his 19th birthday, successfully conned millions of dollars worth of checks as a Pan Am pilot, doctor, and legal prosecutor. An FBI agent makes it his mission to put him behind bars. But Frank not only eludes capture, he revels in the pursuit."
  },
  {
    title: "12 Years a Slave",
    description: "In the pre-Civil War United States, Solomon Northup, a free black man from upstate New York, is abducted and sold into slavery. Facing cruelty as well as unexpected kindnesses Solomon struggles not only to stay alive, but to retain his dignity. In the twelfth year of his unforgettable odyssey, Solomon\u2019s chance meeting with a Canadian abolitionist will forever alter his life."
  },
  {
    title: "La Maison en Petits Cubes",
    description: "La Maison en Petits Cubes tells the story of a grandfather's memories as he adds more blocks to his house to stem the flooding waters."
  },
  {
    title: "The Postman",
    description: "Simple Italian postman learns to love poetry while delivering mail to a famous poet; he uses this to woo local beauty Beatrice."
  },
  {
    title: "The Great Escape",
    description: "The Nazis, exasperated at the number of escapes from their prison camps by a relatively small number of Allied prisoners, relocate them to a high-security 'escape-proof' camp to sit out the remainder of the war. Undaunted, the prisoners plan one of the most ambitious escape attempts of World War II. Based on a true story."
  },
  {
    title: "Close to the Horizon",
    description: "Jessica knows exactly what her life is supposed to look like and where it takes her. But then she meets Danny. He has a complicated past and could confuse all their plans. Jessica has to decide."
  },
  {
    title: "Us Again",
    description: "In a vibrant city pulsating with rhythm and movement, an elderly man and his young-at-heart wife rekindle their youthful passion for life and each other on one magical night."
  },
  {
    title: "Flipped",
    description: "When Juli meets Bryce in the second grade, she knows it's true love. After spending six years trying to convince Bryce the same, she's ready to give up - until he starts to reconsider."
  },
  {
    title: "Invisible Life",
    description: "Rio de Janeiro, Brazil, 1950. In the conservative home of the Gusm\xE3o family, Eur\xEDdice and Guida are two inseparable sisters who support each other. While Guida can share with her younger sister the details of her romantic adventures, Eur\xEDdice finds in her older sister the encouragement she needs to pursue her dream of becoming a professional pianist."
  },
  {
    title: "The Boy Who Harnessed the Wind",
    description: "Against all the odds, a thirteen year old boy in Malawi invents an unconventional way to save his family and village from famine."
  },
  {
    title: "Dallas Buyers Club",
    description: "Loosely based on the true-life tale of Ron Woodroof, a drug-taking, women-loving, homophobic man who in 1986 was diagnosed with HIV/AIDS and given thirty days to live."
  },
  {
    title: "The Sixth Sense",
    description: "Following an unexpected tragedy, a child psychologist named Malcolm Crowe meets an nine year old boy named Cole Sear, who is hiding a dark secret."
  },
  {
    title: "Just Mercy",
    description: "The powerful true story of Harvard-educated lawyer Bryan Stevenson, who goes to Alabama to defend the disenfranchised and wrongly condemned \u2014 including Walter McMillian, a man sentenced to death despite evidence proving his innocence. Bryan fights tirelessly for Walter with the system stacked against them."
  },
  { title: "Tel chi el tel\xF9n", description: "A comedy show." },
  {
    title: "Up",
    description: "Carl Fredricksen spent his entire life dreaming of exploring the globe and experiencing life to its fullest. But at age 78, life seems to have passed him by, until a twist of fate (and a persistent 8-year old Wilderness Explorer named Russell) gives him a new lease on life."
  },
  {
    title: "Teen Titans Go! vs. Teen Titans",
    description: "The comedic modern-day quintet takes on their 2003 counterparts when villains from each of their worlds join forces to pit the two Titan teams against each other. They'll need to set aside their differences and work together to combat Trigon, Hexagon, Santa Claus (that's right, Santa!) and time itself in order to save the multiverse."
  },
  {
    title: "Charm City Kings",
    description: "Mouse desperately wants to join The Midnight Clique, the infamous Baltimore dirt bike riders who rule the summertime streets. When Midnight\u2019s leader, Blax, takes 14-year-old Mouse under his wing, Mouse soon finds himself torn between the straight-and-narrow and a road filled with fast money and violence."
  },
  {
    title: "The Greatest Showman",
    description: "The story of American showman P.T. Barnum, founder of the circus that became the famous traveling Ringling Bros. and Barnum & Bailey Circus."
  },
  {
    title: "Ghost in the Shell",
    description: "In the year 2029, the barriers of our world have been broken down by the net and by cybernetics, but this brings new vulnerability to humans in the form of brain-hacking. When a highly-wanted hacker known as 'The Puppetmaster' begins involving them in politics, Section 9, a group of cybernetically enhanced cops, are called in to investigate and stop the Puppetmaster."
  },
  {
    title: "Mr. Smith Goes to Washington",
    description: "Naive and idealistic Jefferson Smith, leader of the Boy Rangers, is appointed to the United States Senate by the puppet governor of his state. He soon discovers, upon going to Washington, many shortcomings of the political process as his earnest goal of a national boys' camp leads to a conflict with the state political boss."
  },
  {
    title: "The Breadwinner",
    description: "A headstrong young girl in Afghanistan, ruled by the Taliban, disguises herself as a boy in order to provide for her family."
  },
  {
    title: "Rebecca",
    description: "Story of a young woman who marries a fascinating widower only to find out that she must live in the shadow of his former wife, Rebecca, who died mysteriously several years earlier. The young wife must come to grips with the terrible secret of her handsome, cold husband, Max De Winter. She must also deal with the jealous, obsessed Mrs. Danvers, the housekeeper, who will not accept her as the mistress of the house."
  },
  {
    title: "La Grande Vadrouille",
    description: "During World War II, two French civilians and a downed British Bomber Crew set out from Paris to cross the demarcation line between Nazi-occupied Northern France and the South. From there they will be able to escape to England. First, they must avoid German troops - and the consequences of their own blunders."
  }
];

export { movies };
